# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# Aleksandr Brezhnev <abrezhnev@gmail.com>, 2012.
# Alexander Ivanov <hrafn@hrafn.me>, 2012.
# Alexey Vasyukov <alexey.vasyukov@vdel.com>, 2009.
# Alex Nik <rage.iz.me@gmail.com>, 2012.
# Andrew Martynov <andrewm@inventa.ru>, 2004-2006,2008-2009.
# dcantrel <dcantrell@redhat.com>, 2011.
# Dimitris Glezos <glezos@indifex.com>, 2011.
# Inna Kabanova <ikabanova@russianfedora.ru>, 2012.
#   <iya777@yandex.ru>, 2012.
# Leonid Kanter <leon@asplinux.ru>, 2004,2006.
# Misha Shnurapet <shnurapet@fedoraproject.org>, 2011-2012.
# Nikolay Sivov <bunglehead@gmail.com>, 2007.
# Raspopov Sergey <qwerty3456@mail.ru>, 2009.
# Stanislav Darchinov <darchinov@gmail.com>, 2011.
# Stanislav Hanzhin <hanzhin.stas@gmail.com>, 2012.
#   <ti.yarosh@gmail.com>, 2012.
# Valery Suhomlinov <goodguy@goodguy.spb.ru>, 2004.
# wedge <wedge@xwing>, 2006.
# Yulia <ypoyarko@redhat.com>, 2006,2008,2010.
# Yulia  <yulia.poyarkova@redhat.com>, 2006,2008,2010,2012.
# Yuri Khabarov <madb00blik@gmail.com>, 2011.
# Игорь Горбунов <igor.gorbounov@gmail.com>, 2011-2012.
msgid ""
msgstr ""
"Project-Id-Version: Anaconda\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2015-05-19 12:36-0400\n"
"PO-Revision-Date: 2013-01-04 23:55+0000\n"
"Last-Translator: Brian Lane <bcl@redhat.com>\n"
"Language-Team: Russian <trans-ru@lists.fedoraproject.org>\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

#: pyanaconda/bootloader.py:248
#, fuzzy
msgid "/boot file system"
msgstr "файловая система /boot"

#: pyanaconda/bootloader.py:405
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr ""
"Массивы RAID, в которые входит '%s', должны иметь один из следующих raid-"
"уровней: %s."

#: pyanaconda/bootloader.py:413
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_versions)s."
msgstr ""
"Массивы RAID, в которые входит '%s', должны иметь метаданные одной из "
"следующих версий: %s."

#: pyanaconda/bootloader.py:421
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr ""
"Массивы RAID, в которые входит '%s', должны иметь устройства одного из "
"следующих типов: %s."

#: pyanaconda/bootloader.py:437
#, fuzzy, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr "%s должен иметь один из следующих типов меток диска: %s."

#: pyanaconda/bootloader.py:449
#, fuzzy, python-format
msgid "%(desc)s cannot be of type %(type)s."
msgstr "%s не может быть типа %s."

#: pyanaconda/bootloader.py:455
#, fuzzy, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr "%s должен быть смонтирован на одном из %s."

#: pyanaconda/bootloader.py:467
#, fuzzy, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr "%s должен быть размером между %d и %d МБ"

#: pyanaconda/bootloader.py:473
#, fuzzy, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr "%s не должен быть меньше, чем %d МБ."

#: pyanaconda/bootloader.py:482
#, fuzzy, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr "%s не должен быть больше, чем %d МБ."

#: pyanaconda/bootloader.py:500
#, fuzzy, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr "%s должен быть в первых %d МБ диска."

#: pyanaconda/bootloader.py:510
#, python-format
msgid "%s must be on a primary partition."
msgstr "%s должен быть на основном разделе."

#: pyanaconda/bootloader.py:646 pyanaconda/bootloader.py:746
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr "%s не может быть на зашифрованном блочном устройстве."

#: pyanaconda/bootloader.py:701
#, python-format
msgid "%s cannot be on an iSCSI disk on s390(x)"
msgstr ""

#: pyanaconda/bootloader.py:705
#, fuzzy, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr "%s не может быть типа %s"

#: pyanaconda/bootloader.py:1375
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not. A drive failure in %(stage2dev)s "
"could render the system unbootable."
msgstr ""

#: pyanaconda/bootloader.py:1381
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"boot loader will only be installed to a single drive."
msgstr ""

#: pyanaconda/bootloader.py:1669
#, fuzzy, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""
"В %s может быть недостаточно места, чтобы grub2 мог встроить  core.img при "
"использовании файловой системы %s на %s"

#: pyanaconda/constants.py:73
msgid "Start VNC"
msgstr "Запустить VNC"

#: pyanaconda/constants.py:74
msgid "Use text mode"
msgstr "Использовать текстовый режим"

#: pyanaconda/constants.py:138
#, fuzzy
msgid "The password is empty."
msgstr "Пароль установлен."

#: pyanaconda/constants.py:139
#, fuzzy
msgid "The passwords do not match."
msgstr "Пароли не совпадают!"

#: pyanaconda/constants.py:140
msgid "The passwords you entered were different.  Please try again."
msgstr ""

#: pyanaconda/constants.py:141
#, fuzzy, python-format
msgid "The password you have provided is weak. %s"
msgstr "Указанный пароль слишком прост: %s. "

#: pyanaconda/constants.py:142
#, fuzzy, python-format
msgid "The password you have provided is weak: %s. %s"
msgstr "Указанный пароль слишком прост: %s. "

#: pyanaconda/constants.py:143
#, fuzzy
msgid "You have provided a weak password. Press Done again to use anyway."
msgstr ""
"Указанный пароль слишком прост: %s.  Нажмите «Готово» ещё раз, чтобы всё "
"равно его использовать."

#: pyanaconda/constants.py:144
#, fuzzy, python-format
msgid "You have provided a weak password: %s. Press Done again to use anyway."
msgstr ""
"Указанный пароль слишком прост: %s.  Нажмите «Готово» ещё раз, чтобы всё "
"равно его использовать."

#: pyanaconda/constants.py:145
msgid ""
"The password you have provided contains non-ASCII characters. You may not be "
"able to switch between keyboard layouts to login. Press Done to continue."
msgstr ""

#: pyanaconda/constants.py:146
msgid "You will have to press Done twice to confirm it."
msgstr ""

#: pyanaconda/constants.py:148
msgid "Empty"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:125
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:177
msgid "Weak"
msgstr "Простой"

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:128
msgid "Fair"
msgstr "Нормальный"

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:131
msgid "Good"
msgstr "Хороший"

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:134
msgid "Strong"
msgstr "Сложный"

#: pyanaconda/constants_text.py:40 pyanaconda/rescue.py:333
#: pyanaconda/rescue.py:365 pyanaconda/rescue.py:433 pyanaconda/rescue.py:449
#: pyanaconda/rescue.py:460 anaconda:513
msgid "OK"
msgstr "OК"

#: pyanaconda/constants_text.py:44 pyanaconda/text.py:52 pyanaconda/text.py:53
#, fuzzy
msgid "Cancel"
msgstr "_Отмена"

#: pyanaconda/constants_text.py:48
#, fuzzy
msgid "Yes"
msgstr "_Да"

#: pyanaconda/constants_text.py:52
#, fuzzy
msgid "No"
msgstr "_Нет"

#: pyanaconda/errors.py:109
#, fuzzy, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"The installation will now terminate."
msgstr ""
"Следующая ошибка возникла при настройке репозитория установки:\n"
"\n"
"%(e)s\n"
"\n"
"Продолжение установки невозможно."

#: pyanaconda/errors.py:115
#, python-format
msgid "An error occurred while resizing the device %s."
msgstr ""

#: pyanaconda/errors.py:124
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems.  Please check your hardware for the cause of this problem."
msgstr ""

#: pyanaconda/errors.py:133
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or "
"incorrect file system type:\n"
"\n"
msgstr ""

#: pyanaconda/errors.py:139
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error. Would you like to continue "
"using this image?"
msgstr ""

#: pyanaconda/errors.py:153
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""

#: pyanaconda/errors.py:163
#, python-format
msgid "An error occurred mounting the source device %s. Retry?"
msgstr ""

#: pyanaconda/errors.py:171
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: pyanaconda/errors.py:179
#, python-format
msgid ""
"The group '%s' is required for this installation. This group does not exist. "
"This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:185
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does "
"not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:190
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from "
"installation.  This group does not exist.  Would you like to ignore this "
"group and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:202
#, python-format
msgid ""
"The package '%s' is required for this installation. This package does not "
"exist. This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:209
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist.  Would you like to ignore this package and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:220
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is "
"a fatal error and installation will be aborted.  The details of this error "
"are:\n"
"\n"
"%(details)s"
msgstr ""

#: pyanaconda/errors.py:228
#, fuzzy
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""
"Следующая ошибка возникла при настройке репозитория установки:\n"
"\n"
"%(e)s\n"
"\n"
"Продолжение установки невозможно."

#: pyanaconda/errors.py:236
#, fuzzy
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source."
msgstr ""
"Следующее программное обеспечение, отмеченное для установки, содержит "
"ошибки.  Вероятно, это вызвано ошибкой источника установки.  Можно "
"попытаться удалить эти пакеты из установки.\n"
"смените источник установки или выйдите из установщика."

#: pyanaconda/errors.py:244
#, fuzzy
msgid ""
"The following error occurred while installing the boot loader. The system "
"will not be bootable. Would you like to ignore this and continue with "
"installation?"
msgstr ""
"Следующая ошибка возникла при настройке репозитория установки:\n"
"\n"
"%(e)s\n"
"\n"
"Продолжение установки невозможно."

#: pyanaconda/errors.py:256
#, python-format
msgid "Unable to encrypt password: unsupported algorithm %s"
msgstr ""

#: pyanaconda/errors.py:263
#, fuzzy, python-format
msgid ""
"Installation was stopped due to an error installing the boot loader. The "
"exact error message is:\n"
"\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""
"Установка остановлена, по-видимому, из-за проблемы с вашим оборудованием. "
"Сообщение об ошибке в точности:\n"
"\n"
"%s.\n"
"\n"
" Теперь установка будет прекращена."

#: pyanaconda/exception.py:79
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""
"Установка остановлена, по-видимому, из-за проблемы с вашим оборудованием. "
"Сообщение об ошибке в точности:\n"
"\n"
"%s.\n"
"\n"
" Теперь установка будет прекращена."

#: pyanaconda/exception.py:83
#, fuzzy
msgid "Hardware error occured"
msgstr "Возникла аппаратная ошибка"

#: pyanaconda/exception.py:136
#, fuzzy, python-format
msgid ""
"\n"
"The installation was stopped due to incomplete spokes detected while running "
"in non-interactive cmdline mode. Since there cannot be any questions in "
"cmdline mode, edit your kickstart file and retry installation.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""
"Установка остановлена, по-видимому, из-за проблемы с вашим оборудованием. "
"Сообщение об ошибке в точности:\n"
"\n"
"%s.\n"
"\n"
" Теперь установка будет прекращена."

#: pyanaconda/exception.py:144
#, fuzzy, python-format
msgid ""
"\n"
"Running in cmdline mode, no interactive debugging allowed.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""
"Установка остановлена, по-видимому, из-за проблемы с вашим оборудованием. "
"Сообщение об ошибке в точности:\n"
"\n"
"%s.\n"
"\n"
" Теперь установка будет прекращена."

#: pyanaconda/install.py:80
msgid "Configuring installed system"
msgstr ""

#: pyanaconda/install.py:93
msgid "Writing network configuration"
msgstr ""

#: pyanaconda/install.py:97
msgid "Creating users"
msgstr ""

#: pyanaconda/install.py:105
#, fuzzy
msgid "Configuring addons"
msgstr "_Настройки адреса"

#: pyanaconda/install.py:108
msgid "Generating initramfs"
msgstr ""

#: pyanaconda/install.py:119
#, python-format
msgid "Joining realm: %s"
msgstr ""

#: pyanaconda/install.py:122
#, fuzzy
msgid "Running post-installation scripts"
msgstr "Выполняются пред-установочные сценарии"

#: pyanaconda/install.py:170
#, python-format
msgid "Waiting for %s threads to finish"
msgstr ""

#: pyanaconda/install.py:176
#, fuzzy
msgid "Setting up the installation environment"
msgstr "Настройка источника установки..."

#: pyanaconda/install.py:207
msgid "Discovering realm to join"
msgstr ""

#: pyanaconda/install.py:265
#, fuzzy
msgid "Installing boot loader"
msgstr "Устанавливается программное обеспечение"

#: pyanaconda/install.py:268 pyanaconda/packaging/livepayload.py:176
#: pyanaconda/packaging/yumpayload.py:1371
#: pyanaconda/packaging/dnfpayload.py:667
#, fuzzy
msgid "Performing post-installation setup tasks"
msgstr "Выполняются пост-установочные задачи"

#: pyanaconda/iutil.py:811
msgid "No host url"
msgstr ""

#: pyanaconda/iutil.py:832
msgid "malformed URL, cannot parse it."
msgstr ""

#: pyanaconda/iutil.py:848
msgid "URL has no host component"
msgstr ""

#: pyanaconda/kickstart.py:164
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr "Для escrow-сертификата %s требуется сетевое подключение."

#: pyanaconda/kickstart.py:172
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"Произошла ошибка при загрузке сертификата escrow:\n"
"\n"
"%s"

#: pyanaconda/kickstart.py:281
#, fuzzy, python-format
msgid "%s is missing. Cannot setup authentication."
msgstr "Нет %s. Настройка сетевого экрана невозможна."

#: pyanaconda/kickstart.py:306
#, python-format
msgid "autopart fstype of %s is invalid."
msgstr ""

#: pyanaconda/kickstart.py:323
#, python-format
msgid "Settings default fstype to %s failed."
msgstr ""

#: pyanaconda/kickstart.py:354
msgid "GRUB2 does not support installation to a partition."
msgstr ""

#: pyanaconda/kickstart.py:414
#, python-format
msgid "More than one match found for given boot drive \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:417
#, python-format
msgid "Requested boot drive \"%s\" doesn't exist or cannot be used."
msgstr ""

#: pyanaconda/kickstart.py:459
#, python-format
msgid ""
"Btrfs partition \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"btrfs\"."
msgstr ""

#: pyanaconda/kickstart.py:464
#, python-format
msgid "Tried to use undefined partition \"%s\" in Btrfs volume specification."
msgstr ""

#: pyanaconda/kickstart.py:477
msgid ""
"Btrfs volume defined without any member devices.  Either specify member "
"devices or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:486 pyanaconda/kickstart.py:859
#: pyanaconda/kickstart.py:1438
#, python-format
msgid "The mount point \"%s\" is not valid.  It must start with a /."
msgstr ""

#: pyanaconda/kickstart.py:501
#, python-format
msgid "Btrfs volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:587
#, python-format
msgid ""
"Disklabel \"%s\" given in clearpart command is not supported on this "
"platform."
msgstr ""

#: pyanaconda/kickstart.py:599
#, python-format
msgid "Disk \"%s\" given in clearpart command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:612
#, python-format
msgid "Device \"%s\" given in clearpart device list does not exist."
msgstr ""

#: pyanaconda/kickstart.py:639
#, python-format
msgid "NIC \"%s\" given in fcoe command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:690
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr "Нет %s. Настройка сетевого экрана невозможна."

#: pyanaconda/kickstart.py:738 pyanaconda/kickstart.py:749
#, python-format
msgid "Disk \"%s\" given in ignoredisk command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:762
#, python-format
msgid ""
"Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target "
"is not up."
msgstr ""

#: pyanaconda/kickstart.py:772
msgid ""
"iscsi --iface must be specified (binding used) either for all targets or for "
"none"
msgstr ""

#: pyanaconda/kickstart.py:865
#, python-format
msgid ""
"No volume group exists with the name \"%s\".  Specify volume groups before "
"logical volumes."
msgstr ""

#: pyanaconda/kickstart.py:872
#, python-format
msgid ""
"No thin pool exists with the name \"%s\". Specify thin pools before thin "
"volumes."
msgstr ""

#: pyanaconda/kickstart.py:880
msgid "logvol --noformat must also use the --name= option."
msgstr ""

#: pyanaconda/kickstart.py:885 pyanaconda/kickstart.py:946
#, python-format
msgid "Logical volume \"%s\" given in logvol command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:896 pyanaconda/kickstart.py:905
#: pyanaconda/kickstart.py:956 pyanaconda/kickstart.py:1215
#: pyanaconda/kickstart.py:1224 pyanaconda/kickstart.py:1306
#, python-format
msgid "Target size \"%(size)s\" for device \"%(device)s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:919
#, python-format
msgid ""
"Logical volume name \"%(logvol)s\" is already in use in volume group "
"\"%(volgroup)s\"."
msgstr ""

#: pyanaconda/kickstart.py:924
#, python-format
msgid ""
"Logical volume size \"%(logvolSize)s\" must be larger than the volume group "
"extent size of \"%(extentSize)s\"."
msgstr ""

#: pyanaconda/kickstart.py:935 pyanaconda/kickstart.py:1242
#: pyanaconda/kickstart.py:1488
#, fuzzy, python-format
msgid "The \"%s\" file system type is not supported."
msgstr "Данная файловая система не поддерживает метки."

#: pyanaconda/kickstart.py:1029
msgid "No passphrase given for encrypted LV"
msgstr ""

#: pyanaconda/kickstart.py:1083 pyanaconda/kickstart.py:1087
#, python-format
msgid "The %s kickstart command is not currently supported."
msgstr ""

#: pyanaconda/kickstart.py:1112
#, python-format
msgid "No disk found for specified BIOS disk \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1147
#, python-format
msgid "RAID partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1158 pyanaconda/kickstart.py:1413
#, python-format
msgid "PV partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1169 pyanaconda/kickstart.py:1423
#, python-format
msgid "Btrfs partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1192
#, python-format
msgid "The size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1199
msgid "part --noformat must also use the --onpart option."
msgstr ""

#: pyanaconda/kickstart.py:1204 pyanaconda/kickstart.py:1297
#, python-format
msgid "Partition \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1257 pyanaconda/kickstart.py:1271
#, python-format
msgid "Disk \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1260
#, python-format
msgid "Cannot install to unpartitionable device \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1267
#, python-format
msgid "Disk \"%s\" in part command is not partitioned."
msgstr ""

#: pyanaconda/kickstart.py:1280
#, python-format
msgid "The maximum size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1350
msgid "No passphrase given for encrypted part"
msgstr ""

#: pyanaconda/kickstart.py:1445
msgid "raid --noformat must also use the --device option."
msgstr ""

#: pyanaconda/kickstart.py:1450
#, python-format
msgid "RAID device  \"%s\" given in raid command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1471
#, python-format
msgid ""
"RAID device \"%(device)s\" has a format of \"%(format)s\", but should have a "
"format of \"mdmember\"."
msgstr ""

#: pyanaconda/kickstart.py:1476
#, python-format
msgid "Tried to use undefined partition \"%s\" in RAID specification."
msgstr ""

#: pyanaconda/kickstart.py:1504
#, python-format
msgid "RAID volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1511
#, fuzzy, python-format
msgid "The RAID volume name \"%s\" is already in use."
msgstr "Имя %s уже используется."

#: pyanaconda/kickstart.py:1726
#, python-format
msgid ""
"Physical volume \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"lvmpv\"."
msgstr ""

#: pyanaconda/kickstart.py:1731
#, python-format
msgid "Tried to use undefined partition \"%s\" in Volume Group specification"
msgstr ""

#: pyanaconda/kickstart.py:1737
#, python-format
msgid ""
"Volume group \"%s\" defined without any physical volumes.  Either specify "
"physical volumes or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:1747
#, python-format
msgid ""
"Volume group given physical extent size of \"%(extentSize)s\", but must be "
"one of:\n"
"%(validExtentSizes)s."
msgstr ""

#: pyanaconda/kickstart.py:1754
msgid ""
"volgroup --noformat and volgroup --useexisting must also use the --name= "
"option."
msgstr ""

#: pyanaconda/kickstart.py:1759
#, python-format
msgid "Volume group \"%s\" given in volgroup command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1762
#, fuzzy, python-format
msgid "The volume group name \"%s\" is already in use."
msgstr "Группа томов %s уже используется. Изменения не будут сохранены."

#: pyanaconda/kickstart.py:1818
msgid ""
"The upgrade kickstart command is no longer supported. Upgrade functionality "
"is provided through fedup."
msgstr ""

#: pyanaconda/kickstart.py:2092
msgid "Running pre-installation scripts"
msgstr "Выполняются пред-установочные сценарии"

#: pyanaconda/network.py:98
msgid "Host name cannot be None or an empty string."
msgstr ""

#: pyanaconda/network.py:101
#, fuzzy
msgid "Host name must be 255 or fewer characters in length."
msgstr "Имя узла не может содержать более 255 символов."

#: pyanaconda/network.py:104
msgid ""
"Host names can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""

#: pyanaconda/network.py:1325 pyanaconda/ui/gui/spokes/custom.py:545
#: pyanaconda/ui/gui/spokes/network.py:302
#: pyanaconda/ui/gui/spokes/lib/accordion.py:61
msgid "Unknown"
msgstr "Неизвестный"

#: pyanaconda/network.py:1329
msgid "Connecting..."
msgstr "Подключение..."

#: pyanaconda/network.py:1331
msgid "Disconnecting..."
msgstr "Отключение..."

#: pyanaconda/network.py:1351
#, fuzzy, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr "Проводное (%s) подключено"

#: pyanaconda/network.py:1354
#, fuzzy, python-format
msgid "Wireless connected to %(access_point)s"
msgstr "Установлено беспроводное соединение с %s"

#: pyanaconda/network.py:1357
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1361
#, python-format
msgid "Team%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1365
#, fuzzy, python-format
msgid "Bridge%(interface_name)s (%(list_of_slaves)s) connected"
msgstr "Проводное (%s) подключено"

#: pyanaconda/network.py:1371
#, python-format
msgid "VLAN %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:1388
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr ""

#: pyanaconda/network.py:1391
msgid "Not connected"
msgstr "Не подключен"

#: pyanaconda/network.py:1394 pyanaconda/ui/gui/spokes/network.glade:2097
msgid "No network devices available"
msgstr "Сетевые устройства недоступны."

#: pyanaconda/product.py:53
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr ""

#: pyanaconda/product.py:57
#, fuzzy, python-format
msgid "New %(name)s %(version)s Installation"
msgstr "Новая установка %s %s"

#: pyanaconda/rescue.py:184 pyanaconda/rescue.py:350 pyanaconda/rescue.py:426
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr "Выполнить %s для размонтирования системы, когда вы закончите."

#: pyanaconda/rescue.py:187
msgid "When finished please exit from the shell and your system will reboot."
msgstr "После завершения выйдите из оболочки, и система будет перезагружена."

#: pyanaconda/rescue.py:200
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr "Не удаётся найти /bin/sh для запуска! Оболочка не будет запущена."

#: pyanaconda/rescue.py:288 pyanaconda/rescue.py:358 pyanaconda/rescue.py:429
#: pyanaconda/rescue.py:460
msgid "Rescue"
msgstr "Восстановить"

#: pyanaconda/rescue.py:289
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  \n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""
"Режим восстановления сейчас попытается найти вашу систему Linux и "
"смонтировать ее в каталоге %s.  После этого вы сможете внести в систему "
"требуемые изменения.  Если вы хотите продолжить этот шаг, выберите "
"«Продолжить».  Можно также выбрать вариант монтирования файловых систем в "
"режиме только для чтения, а не записи-чтения, выбрав «Только для чтения».  \n"
"\n"
"Если по какой-либо причине этот процесс не удастся, можно выбрать "
"«Пропустить», и этот шаг будет пропущен, а вы попадете прямо в командную "
"оболочку.\n"
"\n"

#: pyanaconda/rescue.py:299
msgid "Continue"
msgstr "Продолжить"

#: pyanaconda/rescue.py:299 pyanaconda/rescue.py:305
msgid "Read-Only"
msgstr "Только для чтения"

#: pyanaconda/rescue.py:299 pyanaconda/rescue.py:301
msgid "Skip"
msgstr "Пропустить"

#: pyanaconda/rescue.py:330
msgid "System to Rescue"
msgstr "Система для восстановления"

#: pyanaconda/rescue.py:331
msgid "Which device holds the root partition of your installation?"
msgstr "Укажите устройство, содержащее корневой раздел для установки."

#: pyanaconda/rescue.py:333 pyanaconda/rescue.py:337
msgid "Exit"
msgstr "Выход"

#: pyanaconda/rescue.py:347 pyanaconda/rescue.py:423
msgid "The system will reboot automatically when you exit from the shell."
msgstr "Система будет перезагружена автоматически после выхода из оболочки."

#: pyanaconda/rescue.py:359
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"
msgstr ""
"Система смонтирована в %(rootPath)s.\n"
"\n"
"Нажмите «Enter» для входа в оболочку. Если вы хотите сделать систему "
"корневой средой, выполните команду:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"

#: pyanaconda/rescue.py:430
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell."
msgstr ""
"При попытке монтирования некоторых из файловых систем возникла ошибка. "
"Некоторые из них могут быть монтированы в %s.\n"
"\n"
"Нажмите «Enter» для входа в оболочку."

#: pyanaconda/rescue.py:438
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "В системе нет разделов Linux. Будет выполнена перезагрузка.\n"

#: pyanaconda/rescue.py:442
msgid " The system will reboot automatically when you exit from the shell."
msgstr "Система будет перезагружена автоматически после выхода из оболочки."

#: pyanaconda/rescue.py:446
msgid "Rescue Mode"
msgstr "Режим восстановления"

#: pyanaconda/rescue.py:447
#, python-format
msgid "You don't have any Linux partitions. Press return to get a shell.%s"
msgstr "В системе нет разделов Linux. Нажмите «Enter» для входа в оболочку.%s"

#: pyanaconda/rescue.py:459
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "Ваша система смонтирована в каталог %s."

#: pyanaconda/storage_utils.py:50 pyanaconda/storage_utils.py:82
#: pyanaconda/ui/gui/spokes/custom.glade:31
msgid "LVM"
msgstr "LVM"

#: pyanaconda/storage_utils.py:51 pyanaconda/storage_utils.py:83
#: pyanaconda/ui/gui/spokes/custom.glade:39
msgid "LVM Thin Provisioning"
msgstr ""

#: pyanaconda/storage_utils.py:52 pyanaconda/ui/gui/spokes/custom.glade:35
msgid "RAID"
msgstr "RAID"

#: pyanaconda/storage_utils.py:53 pyanaconda/storage_utils.py:80
#: pyanaconda/ui/gui/spokes/custom.glade:23
msgid "Standard Partition"
msgstr "Обычный раздел"

#: pyanaconda/storage_utils.py:54 pyanaconda/storage_utils.py:81
#: pyanaconda/ui/gui/spokes/custom.glade:27
msgid "Btrfs"
msgstr ""

#: pyanaconda/storage_utils.py:55
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:713
#: pyanaconda/ui/gui/spokes/lib/resize.glade:152
msgid "Disk"
msgstr "Диск"

#: pyanaconda/storage_utils.py:66
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr "Раздел «swap» используется операционной системой при нехватке памяти."

#: pyanaconda/storage_utils.py:68
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""
"Раздел «boot» используется для хранения файлов, необходимых\n"
"для запуска операционой системы."

#: pyanaconda/storage_utils.py:70
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr "В разделе «root» хранятся файлы приложений и системы."

#: pyanaconda/storage_utils.py:72
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr "В разделе «home» хранятся данные пользователей."

#: pyanaconda/storage_utils.py:74
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""
"Раздел «BIOS boot» требуется для загрузки системы с дисков\n"
"с разметкой GPT на оборудовании с BIOS."

#: pyanaconda/storage_utils.py:76
#, fuzzy
msgid ""
"The PReP boot partition is required as part of the\n"
"boot loader configuration on some PPC platforms."
msgstr ""
"На некоторых платформах PPC для размещения\n"
"конфигурации загрузчика требуется раздел PReP."

#: pyanaconda/storage_utils.py:184
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"Размер корневого раздела меньше 250 МБ. Как правило, этого недостаточно для "
"установки %s."

#: pyanaconda/storage_utils.py:189
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""
"Вы не указали корневой раздел (/), который необходим для продолжения "
"установки %s."

#: pyanaconda/storage_utils.py:203
msgid ""
"This platform requires /boot on a dedicated partition or logical volume.  If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""
"Для данной платформы требуется монтировать /boot на выделенном разделе или "
"логическом томе. Если вы не хотите выносить /boot на отдельный том, "
"необходимо разместить / на выделенном не-LVM разделе."

#: pyanaconda/storage_utils.py:213
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""
"Размер раздела %(mount)s меньше %(size)s МБ, что меньше рекомендуемого "
"размера для нормальной установки %(productName)s. "

#: pyanaconda/storage_utils.py:223
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""
"Раздел %(mount)s слишком мал для %(format)s форматирования (допустимые "
"размеры: от %(minSize)d МБ до %(maxSize)d МБ)"

#: pyanaconda/storage_utils.py:229
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""
"Раздел %(mount)s слишком велик для %(format)s форматирования (допустимые "
"размеры: от %(minSize)d МБ до %(maxSize)d МБ)"

#: pyanaconda/storage_utils.py:238
#, fuzzy
msgid "No valid boot loader target device found. See below for details."
msgstr "Не удалось добавить новое устройство. Подробнее…"

#: pyanaconda/storage_utils.py:250
msgid "You have not created a bootable partition."
msgstr "Вы не создали загружаемый раздел."

#: pyanaconda/storage_utils.py:272
#, fuzzy
msgid ""
"Your BIOS-based system needs a special partition to boot from a GPT disk "
"label. To continue, please create a 1MiB 'biosboot' type partition."
msgstr ""
"BIOS в вашей системе требует специальный раздел для загрузки %s с новым "
"форматом меток диска (GPT). Для продолжения создайте раздел типа 'BIOS Boot' "
"размером 1 МБ."

#: pyanaconda/storage_utils.py:283
#, fuzzy, python-format
msgid ""
"You have not specified a swap partition.  %(requiredMem)s of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""
"Вы не определили swap-раздел.  Для продолжения установки без swap-раздела "
"необходимо %(requiredMem)s МБ памяти, а доступно лишь %(installedMem)s МБ."

#: pyanaconda/storage_utils.py:290
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"Вы не создали раздел подкачки. Хотя это и не всегда является необходимым, в "
"большинстве случаев раздел подкачки может значительно улучшить "
"производительность системы."

#: pyanaconda/storage_utils.py:297
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""
"По крайней мере одно из устройств подкачки не имеет UUID, что обычно бывает, "
"если область подкачки создается с использованием старых версий mkswap. К "
"этим устройствам обращение будет по пути к устройству из /etc/fstab, а это "
"не самый лучший способ, так как пути устройств могут меняться при различных "
"обстоятельствах. "

#: pyanaconda/storage_utils.py:308
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""
"Неверная точка монтирования. Каталог %s должен располагаться в файловой "
"системе /."

#: pyanaconda/storage_utils.py:313
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "Точка монтирования %s должна располагаться в файловой системе Linux. "

#: pyanaconda/storage_utils.py:341
#, python-format
msgid ""
"Encryption requested for LUKS device %s but no encryption key specified for "
"this device."
msgstr ""

#: pyanaconda/text.py:99
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: pyanaconda/text.py:106
#, fuzzy
msgid "Passphrase"
msgstr "Пароль:"

#: pyanaconda/vnc.py:74
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr "%(productName)s %(productVersion)s установка"

#: pyanaconda/vnc.py:135
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr "Установка %(productName)s %(productVersion)s на %(name)s"

#: pyanaconda/vnc.py:154
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Попытка соединения с VNC-клиентом на узле %s..."

#: pyanaconda/vnc.py:168
msgid "Connected!"
msgstr "Подключен!"

#: pyanaconda/vnc.py:171
msgid "Will try to connect again in 15 seconds..."
msgstr ""
"Повторная попытка установить соединение будет предпринята через 15 секунд..."

#: pyanaconda/vnc.py:178
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "Установка соединения прекращена после %d попытки.\n"
msgstr[1] "Установка соединения прекращена после %d попыток.\n"
msgstr[2] "Установка соединения прекращена после %d попыток.\n"

#: pyanaconda/vnc.py:186
msgid "Attempting to start vncconfig"
msgstr ""

#: pyanaconda/vnc.py:199
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr "Для начала установки подключите VNC-клиент к %s вручную."

#: pyanaconda/vnc.py:201
#, python-format
msgid ""
"Please manually connect your vnc client to <IP ADDRESS>:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the <IP "
"ADDRESS>."
msgstr ""

#: pyanaconda/vnc.py:206
msgid "Starting VNC..."
msgstr "Запуск VNC..."

#: pyanaconda/vnc.py:243
msgid "The VNC server is now running."
msgstr "Сервер VNC запущен."

#: pyanaconda/vnc.py:247
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Вы выбрали подключение к прослушивающему VNC-клиенту. \n"
"Этот режим не требует указания пароля. Если вы\n"
"определите пароль, его можно будет использовать для \n"
"подключения к VNC-серверу в том случае, если тому не\n"
"удастся подключиться к прослушивающему VNC-клиенту.\n"
"\n"

#: pyanaconda/vnc.py:252
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"Внимание! Сервер VNC запущен БЕЗ ПАРОЛЯ!\n"
"Используйте параметр загрузки vncpassword=«пароль»,\n"
"если хотите защитить доступ к серверу.\n"
"\n"

#: pyanaconda/vnc.py:256
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Вы выбрали запуск VNC с паролем.\n"
"\n"

#: pyanaconda/vnc.py:258
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Неизвестная ошибка. Остановка работы.\n"
"\n"

#: pyanaconda/vnc.py:280
#, fuzzy
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""
"Указанный пароль VNC содержит меньше 6 символов.\n"
"Введите новый пароль. Для работы без пароля оставьте поле пустым."

#: pyanaconda/packaging/__init__.py:1113
#, fuzzy
msgid "Failed to set up installation source"
msgstr "Не удалось настроить источник для установки"

#: pyanaconda/packaging/__init__.py:1114
#, fuzzy
msgid "Error downloading package metadata"
msgstr "Ошибка загрузки метаданных пакета..."

#: pyanaconda/packaging/__init__.py:1115
msgid "No installation source available"
msgstr "Нет доступных источников для установки"

#: pyanaconda/packaging/livepayload.py:104
#: pyanaconda/packaging/livepayload.py:125
msgid "Installing software"
msgstr "Устанавливается программное обеспечение"

#: pyanaconda/packaging/livepayload.py:237
#: pyanaconda/packaging/livepayload.py:246
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr ""

#: pyanaconda/packaging/livepayload.py:375
msgid "Checking image checksum"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1186
#, python-format
msgid "CmdlineError: Missing package: %s"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1331
#: pyanaconda/packaging/dnfpayload.py:621
msgid "Starting package installation process"
msgstr "Начинается процесс установки пакетов"

#: pyanaconda/packaging/yumpayload.py:1369
#: pyanaconda/packaging/dnfpayload.py:650
msgid "Preparing transaction from installation source"
msgstr "Подготовка транзакции из источника установки"

#: pyanaconda/packaging/yumpayload.py:1370
msgid "Installing"
msgstr "Установка"

#: pyanaconda/packaging/dnfpayload.py:146
#, python-format
msgid ""
"Downloading %(total_files)s RPMs, %(downloaded)s / %(total_size)s "
"(%(percent)d%%) done."
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:638
#, fuzzy
msgid "Downloading packages"
msgstr "Загрузка метаданных пакета..."

#: pyanaconda/packaging/dnfpayload.py:660
#, fuzzy, python-format
msgid "Installing %s"
msgstr "Установка"

#: pyanaconda/packaging/rpmostreepayload.py:163
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:179
#, fuzzy, python-format
msgid "Preparing deployment of %s"
msgstr "Подготовка к установке"

#: pyanaconda/packaging/rpmostreepayload.py:199
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:202
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/ui/helpers.py:94 pyanaconda/ui/tui/spokes/storage.py:389
msgid "Checking storage configuration..."
msgstr "Проверка настроек накопителей..."

#: pyanaconda/ui/categories/__init__.py:48
msgid "DEFAULT TITLE"
msgstr "НАЗВАНИЕ ПО УМОЛЧАНИЮ"

#: pyanaconda/ui/categories/customization.py:32
#, fuzzy
msgid "CUSTOMIZATION"
msgstr "РЕГИОНАЛЬНЫЕ НАСТРОЙКИ"

#: pyanaconda/ui/categories/localization.py:31
msgid "LOCALIZATION"
msgstr "РЕГИОНАЛЬНЫЕ НАСТРОЙКИ"

#: pyanaconda/ui/categories/software.py:31
msgid "SOFTWARE"
msgstr "ПРОГРАММНОЕ ОБЕСПЕЧЕНИЕ"

#: pyanaconda/ui/categories/system.py:32
#: pyanaconda/ui/gui/spokes/lib/accordion.py:166
msgid "SYSTEM"
msgstr "СИСТЕМА"

#: pyanaconda/ui/categories/user_settings.py:32
msgid "USER SETTINGS"
msgstr ""

#: pyanaconda/ui/lib/space.py:37
#, python-format
msgid ""
"Not enough space in file systems for the current software selection. An "
"additional %s is needed."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:58
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:67
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:68
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of "
"available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:92
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:95
#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:88
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: pyanaconda/ui/lib/entropy.py:102
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:107
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:110
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:35
#, fuzzy
msgid "Installation"
msgstr "Узел установки"

#: pyanaconda/ui/tui/hubs/summary.py:51
msgid "Starting automated install"
msgstr "Запуск автоматической установки..."

#: pyanaconda/ui/tui/hubs/summary.py:85
msgid "The following mandatory spokes are not completed:"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:93
#, fuzzy
msgid ""
"  Please make your choice from above ['q' to quit | 'b' to begin "
"installation |\n"
"  'r' to refresh]: "
msgstr "Выберите ['q' для выхода | 'c' для продолжения]: "

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:107
msgctxt "TUI|Spoke Navigation"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:110 pyanaconda/ui/tui/hubs/__init__.py:114
msgid "Please complete all spokes before continuing"
msgstr "Заполните все поля, прежде чем продолжить"

#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/summary.py:121 pyanaconda/ui/tui/hubs/__init__.py:111
#: pyanaconda/ui/tui/simpleline/base.py:456
#: pyanaconda/ui/tui/spokes/software.py:189
#, fuzzy
msgctxt "TUI|Spoke Navigation"
msgid "c"
msgstr "c"

#: pyanaconda/ui/tui/hubs/__init__.py:41
msgid "Default HUB title"
msgstr "Название узла по умолчанию"

#: pyanaconda/ui/tui/tuiobject.py:30 pyanaconda/ui/gui/main.glade:72
msgid "Error"
msgstr "Ошибка"

#: pyanaconda/ui/tui/tuiobject.py:51
msgid "Press enter to exit."
msgstr "Нажмите Enter для выхода."

#: pyanaconda/ui/tui/tuiobject.py:61
msgid "Question"
msgstr "Вопрос"

#: pyanaconda/ui/tui/tuiobject.py:84
msgid "Please respond 'yes' or 'no': "
msgstr "Отвечайте 'да' или 'нет': "

#: pyanaconda/ui/tui/tuiobject.py:87
msgid "yes"
msgstr "да"

#: pyanaconda/ui/tui/tuiobject.py:92
msgid "no"
msgstr "нет"

#: pyanaconda/ui/tui/simpleline/base.py:87
msgid "Do you really want to quit?"
msgstr "Действительно хотите выйти?"

#. TRANSLATORS: 'r' to refresh
#: pyanaconda/ui/tui/simpleline/base.py:450
msgctxt "TUI|Spoke Navigation"
msgid "r"
msgstr ""

#. TRANSLATORS: 'q' to quit
#: pyanaconda/ui/tui/simpleline/base.py:462
#, fuzzy
msgctxt "TUI|Spoke Navigation"
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/simpleline/base.py:571
msgid "Press ENTER to continue"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:621
#, fuzzy
msgid ""
"  Please make your choice from above ['q' to quit | 'c' to continue |\n"
"  'r' to refresh]: "
msgstr "Выберите ['q' для выхода | 'c' для продолжения]: "

#: pyanaconda/ui/tui/spokes/askvnc.py:42
msgid "VNC"
msgstr "VNC"

#: pyanaconda/ui/tui/spokes/askvnc.py:58
#, fuzzy
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""
"Невозможно запустить графическую подсистему X на вашей машине. Запустить VNC "
"для подключения к данному компьютеру с другой машины и выполнения установки "
"в графическом режиме или продолжить установку в текстовом режиме?"

#: pyanaconda/ui/tui/spokes/askvnc.py:106
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/spokes/askvnc.py:122
msgid "VNC Password"
msgstr "Пароль VNC"

#: pyanaconda/ui/tui/spokes/askvnc.py:130
#, fuzzy
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""
"Введите пароль VNC. Его потребуется напечатать дважды.\n"
"Пустой ввод означает работу без пароля."

#: pyanaconda/ui/tui/spokes/askvnc.py:149
#: pyanaconda/ui/gui/spokes/network.py:1319
msgid "Password: "
msgstr "Пароль:"

#: pyanaconda/ui/tui/spokes/askvnc.py:150
msgid "Password (confirm): "
msgstr "Пароль (подтверждение):"

#: pyanaconda/ui/tui/spokes/askvnc.py:153
msgid "Passwords do not match!"
msgstr "Пароли не совпадают!"

#: pyanaconda/ui/tui/spokes/askvnc.py:156
msgid "The password must be at least six characters long."
msgstr "Пароль должен содержать не менее 6 символов."

#: pyanaconda/ui/tui/spokes/askvnc.py:160
#, fuzzy
msgid "The password cannot be more than eight characters long."
msgstr "Пароль должен содержать не менее 6 символов."

#: pyanaconda/ui/tui/spokes/langsupport.py:38
#, fuzzy
msgid "Language settings"
msgstr "Настройка временных зон"

#: pyanaconda/ui/tui/spokes/langsupport.py:65
#, fuzzy
msgid "Language is not set."
msgstr "Временная зона не задана"

#: pyanaconda/ui/tui/spokes/langsupport.py:75
#, fuzzy
msgid "Available locales"
msgstr "Доступные регионы"

#: pyanaconda/ui/tui/spokes/langsupport.py:78
#, fuzzy
msgid "Available languages"
msgstr "Доступные регионы"

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/langsupport.py:112
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:120
msgid ""
"Please select language support to install.\n"
"[b to language list, c to continue, q to quit]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:43
#, fuzzy
msgid "Network configuration"
msgstr "Проверка настроек накопителей..."

#: pyanaconda/ui/tui/spokes/network.py:94
#, fuzzy, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr "Проводное (%s) подключено"

#: pyanaconda/ui/tui/spokes/network.py:99
#, fuzzy, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr "Проводное (%s) подключено"

#: pyanaconda/ui/tui/spokes/network.py:111
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:113
#, python-format
msgid " DNS: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:120
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:140
#, fuzzy, python-format
msgid "Host Name: %s\n"
msgstr "Имя сервера"

#: pyanaconda/ui/tui/spokes/network.py:152
#, fuzzy
msgid "Set host name"
msgstr "Имя сервера"

#: pyanaconda/ui/tui/spokes/network.py:154
#, python-format
msgid "Configure device %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:173
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:241
#: pyanaconda/ui/gui/spokes/network.glade:2233
#, fuzzy
msgid "Host Name"
msgstr "Имя сервера"

#: pyanaconda/ui/tui/spokes/network.py:191
msgid ""
"Configuration not saved: gateway or netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:207
msgid "Can't apply configuration, device activation failed."
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:234
#: pyanaconda/ui/gui/spokes/network.py:1466
#: pyanaconda/ui/gui/spokes/network.py:1535
#, fuzzy, python-format
msgid "Host name is not valid: %s"
msgstr "Имя сервера"

#: pyanaconda/ui/tui/spokes/network.py:258
#, fuzzy
msgid "Device configuration"
msgstr "Завершить настройку"

#: pyanaconda/ui/tui/spokes/network.py:262
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:264
msgid "IPv4 netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:265
#, fuzzy
msgid "IPv4 gateway"
msgstr "Шлюз"

#: pyanaconda/ui/tui/spokes/network.py:266
#, python-format
msgid ""
"IPv6 address or %(auto)s for automatic, %(dhcp)s for DHCP, %(ignore)s to "
"turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:269
msgid "IPv6 default gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:270
msgid "Nameservers (comma separated)"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:271
msgid "Connect automatically after reboot"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:272
#, fuzzy
msgid "Apply configuration in installer"
msgstr "в описании файловой системы отсутствует поле тип"

#: pyanaconda/ui/tui/spokes/network.py:287
#, python-format
msgid "Configuring device %s."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:32
#, fuzzy
msgid "Root password"
msgstr "Пароль root:"

#: pyanaconda/ui/tui/spokes/password.py:55
msgid "Password is set."
msgstr "Пароль установлен."

#: pyanaconda/ui/tui/spokes/password.py:57
msgid "Root account is disabled."
msgstr "Пользователь root заблокирован."

#: pyanaconda/ui/tui/spokes/password.py:59
msgid "Password is not set."
msgstr "Пароль не установлен."

#: pyanaconda/ui/tui/spokes/password.py:64
msgid "Please select new root password. You will have to type it twice."
msgstr "Выберите новый пароль root. Его потребуется ввести дважды."

#: pyanaconda/ui/tui/spokes/password.py:70
#: pyanaconda/ui/gui/spokes/password.glade:86
#: pyanaconda/ui/gui/spokes/user.glade:170
#, fuzzy
msgid "Password"
msgstr "Пароль:"

#: pyanaconda/ui/tui/spokes/storage.py:55
msgid "Use All Space"
msgstr "Использовать все пространство"

#: pyanaconda/ui/tui/spokes/storage.py:56
msgid "Replace Existing Linux system(s)"
msgstr "Заменить существующую систему(мы) Linux"

#: pyanaconda/ui/tui/spokes/storage.py:57
msgid "Use Free Space"
msgstr "Использовать свободное пространство"

#: pyanaconda/ui/tui/spokes/storage.py:67
#, fuzzy
msgid "Installation Destination"
msgstr "Место установки"

#: pyanaconda/ui/tui/spokes/storage.py:123
#: pyanaconda/ui/gui/spokes/storage.py:380
msgid "No disks selected"
msgstr "Ни одного диска не выбрано"

#: pyanaconda/ui/tui/spokes/storage.py:126
#: pyanaconda/ui/gui/spokes/storage.py:383
msgid "Kickstart insufficient"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:133
#: pyanaconda/ui/gui/spokes/storage.py:392
msgid "Error checking storage configuration"
msgstr "Ошибка при  проверке настроек накопителей"

#: pyanaconda/ui/tui/spokes/storage.py:135
#: pyanaconda/ui/gui/spokes/storage.py:394
#, fuzzy
msgid "Warning checking storage configuration"
msgstr "Ошибка при  проверке настроек накопителей"

#: pyanaconda/ui/tui/spokes/storage.py:138
#: pyanaconda/ui/gui/spokes/storage.py:396
msgid "Automatic partitioning selected"
msgstr "Выбрано автоматическое разбиение диска"

#: pyanaconda/ui/tui/spokes/storage.py:140
#: pyanaconda/ui/gui/spokes/storage.py:398
msgid "Custom partitioning selected"
msgstr "Выбрано собственное разбиение диска"

#: pyanaconda/ui/tui/spokes/storage.py:176
#: pyanaconda/ui/gui/spokes/storage.py:649
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""
"Ни один диск не обнаружен. Выключите компьютер и подключите хотя бы один "
"диск. После этого можно вновь начать установку."

#: pyanaconda/ui/tui/spokes/storage.py:178
#: pyanaconda/ui/gui/spokes/storage.py:651
msgid "No disks selected; please select at least one disk to install to."
msgstr "Ни один диск не выбран; необходим хотя бы один диск для установки."

#: pyanaconda/ui/tui/spokes/storage.py:193
#: pyanaconda/ui/tui/spokes/source.py:366
#: pyanaconda/ui/gui/spokes/source.py:657
#: pyanaconda/ui/gui/spokes/storage.py:600
msgid "Probing storage..."
msgstr "Проверка устройств хранения..."

#: pyanaconda/ui/tui/spokes/storage.py:214
msgid "Select all"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:307
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:309
msgid ""
"Warning: All storage changes made using the installer will be lost when you "
"choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:324
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.py:92
#, python-format
msgid "Formatting /dev/%s. This may take a moment."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:368
msgid "Generating updated storage configuration"
msgstr "Создание обновленной конфигурации устройств хранения"

#: pyanaconda/ui/tui/spokes/storage.py:373
#: pyanaconda/ui/tui/spokes/storage.py:385
#, python-format
msgid "storage configuration failed: %s"
msgstr "Ошибка настройки хранилища: %s"

#: pyanaconda/ui/tui/spokes/storage.py:432
msgid "Autopartitioning Options"
msgstr "Настройки автоматического создания разделов"

#: pyanaconda/ui/tui/spokes/storage.py:458
#, fuzzy
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target."
msgstr ""
"Для установки необходимо разбиение жесткого диска на разделы.  Выберите, "
"сколько места потребуется для установки целевой системы."

#: pyanaconda/ui/tui/spokes/storage.py:497
#, fuzzy
msgid "Partition Scheme Options"
msgstr "Конфигурация схемы разделов"

#: pyanaconda/ui/tui/spokes/storage.py:521
#, fuzzy
msgid "Select a partition scheme configuration."
msgstr "Конфигурация схемы разделов"

#: pyanaconda/ui/tui/spokes/time_spoke.py:31
msgid "Timezone settings"
msgstr "Настройка временных зон"

#: pyanaconda/ui/tui/spokes/time_spoke.py:61
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:533
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:539
#, python-format
msgid "%s timezone"
msgstr "Часовой пояс %s"

#: pyanaconda/ui/tui/spokes/time_spoke.py:63
msgid "Timezone is not set."
msgstr "Временная зона не задана"

#: pyanaconda/ui/tui/spokes/time_spoke.py:71
#, python-format
msgid "Available timezones in region %s"
msgstr "Доступные временные зоны в регионе %s"

#: pyanaconda/ui/tui/spokes/time_spoke.py:74
msgid "Available regions"
msgstr "Доступные регионы"

#: pyanaconda/ui/tui/spokes/time_spoke.py:133
msgid ""
"Please select the timezone.\n"
"Use numbers or type names directly [b to region list, q to quit]: "
msgstr ""
"Выберите временную зону.\n"
"Используйте цифры или непосредственно введите название [b для списка "
"регионов, q для выхода]: "

#: pyanaconda/ui/tui/spokes/shell_spoke.py:33
msgid "Shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:48
#, fuzzy
msgid "Start shell"
msgstr "Запустить VNC"

#: pyanaconda/ui/tui/spokes/shell_spoke.py:57
msgid "Exit the shell to continue"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:54
#, fuzzy
msgid "Installation source"
msgstr "Прогресс установки"

#: pyanaconda/ui/tui/spokes/source.py:57 pyanaconda/ui/tui/spokes/source.py:106
#: pyanaconda/ui/gui/spokes/source.py:547
#: pyanaconda/ui/gui/spokes/source.glade:757
msgid "Closest mirror"
msgstr "Ближайшее зеркало"

#: pyanaconda/ui/tui/spokes/source.py:98 pyanaconda/ui/gui/spokes/source.py:539
#, python-format
msgid "NFS server %s"
msgstr "Сервер NFS %s"

#: pyanaconda/ui/tui/spokes/source.py:100
#: pyanaconda/ui/gui/spokes/source.py:541
msgid "Local media"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:103
#: pyanaconda/ui/tui/spokes/source.py:117
#: pyanaconda/ui/gui/spokes/source.py:535
msgid "Error setting up software source"
msgstr "Ошибка при настройке источника программного обеспечения"

#: pyanaconda/ui/tui/spokes/source.py:108
#: pyanaconda/ui/tui/spokes/software.py:121
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:541
#: pyanaconda/ui/gui/spokes/software.py:224
#: pyanaconda/ui/gui/spokes/source.py:549
msgid "Nothing selected"
msgstr "Ничего не выбрано"

#: pyanaconda/ui/tui/spokes/source.py:119
#: pyanaconda/ui/tui/spokes/software.py:109
#, fuzzy
msgid "Processing..."
msgstr "Подключение..."

#: pyanaconda/ui/tui/spokes/source.py:135
#, fuzzy
msgid "CD/DVD"
msgstr "Привод CD/DVD"

#: pyanaconda/ui/tui/spokes/source.py:135
#, fuzzy
msgid "local ISO file"
msgstr "_ISO-файл:"

#: pyanaconda/ui/tui/spokes/source.py:135
#, fuzzy
msgid "Network"
msgstr "Сетевое время"

#: pyanaconda/ui/tui/spokes/source.py:139
#: pyanaconda/ui/gui/spokes/source.py:850
msgid ""
"The installation source is in use by the installer and cannot be changed."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:146
#, fuzzy
msgid "Choose an installation source type."
msgstr "Нет доступных источников для установки"

#: pyanaconda/ui/tui/spokes/source.py:243
#: pyanaconda/ui/tui/spokes/source.py:281
msgid "Specify Repo Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:247
msgid "Repo URL"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:285
#, fuzzy
msgid "<server>:/<path>"
msgstr "Сервер NFS %s"

#: pyanaconda/ui/tui/spokes/source.py:286
#, fuzzy
msgid "NFS mount options"
msgstr "Параметры монтирования NFS:"

#: pyanaconda/ui/tui/spokes/source.py:328
#, fuzzy
msgid "Select device containing the ISO file"
msgstr "устройство, содержащее /boot"

#: pyanaconda/ui/tui/spokes/source.py:387
msgid "No mountable devices found"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:415
#, fuzzy
msgid "Select an ISO to use as install source"
msgstr "Не удалось настроить источник для установки"

#: pyanaconda/ui/tui/spokes/source.py:444
msgid "No *.iso files found in device root folder"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:40
#, fuzzy
msgid "Software selection"
msgstr "_Изменить выбор программного обеспечения"

#: pyanaconda/ui/tui/spokes/software.py:107
#: pyanaconda/ui/gui/spokes/software.py:208
msgid "Error checking software selection"
msgstr "Ошибка при проверке выбора программного обеспечения"

#: pyanaconda/ui/tui/spokes/software.py:111
#: pyanaconda/ui/gui/spokes/software.py:211
msgid "Installation source not set up"
msgstr "Источник установки не настроен"

#: pyanaconda/ui/tui/spokes/software.py:113
#: pyanaconda/ui/gui/spokes/software.py:214
msgid "Source changed - please verify"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:120
#: pyanaconda/ui/gui/spokes/software.py:222
msgid "Custom software selected"
msgstr "Свой выбор программного обеспечения"

#: pyanaconda/ui/tui/spokes/software.py:147
#, fuzzy
msgid "Installation source needs to be set up first."
msgstr "Источник установки не настроен"

#: pyanaconda/ui/tui/spokes/software.py:157
#, fuzzy
msgid "Base environment"
msgstr "Выберите среду"

#: pyanaconda/ui/tui/spokes/user.py:37
#, fuzzy
msgid "User creation"
msgstr "_Использовать проверку подлинности"

#: pyanaconda/ui/tui/spokes/user.py:115 pyanaconda/ui/gui/spokes/user.py:362
msgid "No user will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:117
#, fuzzy
msgid "You must set a password"
msgstr "Необходимо ввести и подтвердить пароль."

#: pyanaconda/ui/tui/spokes/user.py:119 pyanaconda/ui/gui/spokes/user.py:364
#, python-format
msgid "Administrator %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:121 pyanaconda/ui/gui/spokes/user.py:366
#, python-format
msgid "User %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings.py:36
#, fuzzy
msgid "Warnings"
msgstr "Внимание"

#: pyanaconda/ui/tui/spokes/warnings.py:44
#: pyanaconda/ui/gui/spokes/welcome.glade:517
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: pyanaconda/ui/tui/spokes/progress.py:37
#, fuzzy
msgid "Progress"
msgstr "В процессе"

#: pyanaconda/ui/tui/spokes/progress.py:139
msgid "\tInstallation complete.  Press return to quit"
msgstr "\tУстановка завершена.  Нажмите Enter для выхода"

#: pyanaconda/ui/tui/spokes/__init__.py:49
msgid "Default spoke title"
msgstr "Стандартное название для запоминания"

#: pyanaconda/ui/tui/spokes/__init__.py:61
msgid "testing status..."
msgstr "проверка состояния..."

#: pyanaconda/ui/tui/spokes/__init__.py:103
#, fuzzy
msgid "New value"
msgstr "Новое устройство"

#: pyanaconda/ui/tui/spokes/__init__.py:128
#, python-format
msgid "%s: "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:129
#, fuzzy, python-format
msgid "%s (confirm): "
msgstr "Пароль (подтверждение):"

#: pyanaconda/ui/tui/spokes/__init__.py:132
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:154
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""
"\n"
"Хотите всё равно его использовать?"

#: pyanaconda/ui/tui/spokes/__init__.py:171
msgid ""
"You have provided a password containing non-ASCII characters.\n"
"You may not be able to switch between keyboard layouts to login.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:177
#, python-format
msgid "Enter new value for '%s' and press enter\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:185
#, python-format
msgid ""
"You have provided an invalid user name: %s\n"
"Tip: Keep your user name shorter than 32 characters and do not use spaces.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:296
#, fuzzy
msgid "Password set."
msgstr "Пароль установлен."

#: pyanaconda/ui/gui/__init__.py:334 widgets/src/BaseWindow.c:115
msgid "Help!"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:752 widgets/src/StandaloneWindow.c:51
msgid "_Quit"
msgstr "В_ыход "

#: pyanaconda/ui/gui/__init__.py:767
msgid "_No"
msgstr "_Нет"

#: pyanaconda/ui/gui/__init__.py:767
msgid "_Yes"
msgstr "_Да"

#: pyanaconda/ui/gui/hubs/__init__.py:224
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""
"Заполните элементы, отмеченные этим значком, прежде чем перейти к следующему "
"шагу."

#: pyanaconda/ui/gui/hubs/progress.py:111
msgid "Complete!"
msgstr "Готово!"

#: pyanaconda/ui/gui/hubs/progress.py:211
#, fuzzy, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"When you are ready, reboot your system to start using it!"
msgstr ""
"%s успешно установлена на компьютер и готова\n"
"к использованию!  Перезагрузите систему и начните использовать её!"

#: pyanaconda/ui/gui/hubs/progress.py:214
#, fuzzy
msgctxt "GUI|Progress"
msgid "_Quit"
msgstr "В_ыход "

#: pyanaconda/ui/gui/hubs/progress.py:222
#, fuzzy, python-format
msgid ""
"%s is now successfully installed, but some configuration still needs to be "
"done.\n"
"Finish it and then click the Finish configuration button please."
msgstr ""
"%s успешно установлена на ваш компьютер, но требуется выполнить некоторую "
"настройку.\n"
"Завершите её и нажмите на кнопку «Завершить настройку»."

#: pyanaconda/ui/gui/hubs/progress.py:227
#, fuzzy, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and reboot to start using it!"
msgstr ""
"%s успешно установлена на компьютер и готова\n"
"к использованию!  Перезагрузите систему и начните использовать её!"

#: pyanaconda/ui/gui/spokes/custom.py:107
#, fuzzy, python-format
msgid "Create a new %(container_type)s ..."
msgstr "Создать группу томов..."

#: pyanaconda/ui/gui/spokes/custom.py:108
#, python-format
msgid "Create or select %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:110
msgid "Device reconfiguration failed. Click for details."
msgstr "Перенастройка устройства не удалась. Подробнее…"

#: pyanaconda/ui/gui/spokes/custom.py:112
msgid ""
"Storage configuration reset due to unrecoverable error. Click for details."
msgstr "Настройки хранилища сброшены из-за неустранимой ошибки. Подробнее…"

#: pyanaconda/ui/gui/spokes/custom.py:143
#: pyanaconda/ui/gui/spokes/custom.glade:93
msgid "MANUAL PARTITIONING"
msgstr "РАЗМЕТКА ВРУЧНУЮ"

#: pyanaconda/ui/gui/spokes/custom.py:369
#, fuzzy, python-format
msgctxt "GUI|Custom Partitioning"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] "%d накопитель выбран"
msgstr[1] "%d накопителя выбрано"
msgstr[2] "%d накопителей выбрано"

#: pyanaconda/ui/gui/spokes/custom.py:495
#, fuzzy, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""
"При создании точек монтирования для установки %s %s\n"
"можно просмотреть сведения о них здесь."

#: pyanaconda/ui/gui/spokes/custom.py:655
#, fuzzy, python-format
msgid ""
"/boot/efi must be on a device of type %(oneFsType)s or %(anotherFsType)s"
msgstr "/boot/efi должен быть на устройстве типа %s"

#: pyanaconda/ui/gui/spokes/custom.py:659
#, fuzzy, python-format
msgid "%(fs)s must be on a device of type %(type)s"
msgstr "%s должен быть на устройстве типа %s"

#: pyanaconda/ui/gui/spokes/custom.py:662
#: pyanaconda/ui/gui/spokes/custom.py:664
#, python-format
msgid "%s cannot be encrypted"
msgstr "%s не может быть зашифрован"

#: pyanaconda/ui/gui/spokes/custom.py:666
#, fuzzy
msgid "You must create a new file system on the root device."
msgstr "Вы должны создать новую файловую систему на корневом устройстве."

#: pyanaconda/ui/gui/spokes/custom.py:671
#, fuzzy, python-format
msgid "Device does not support RAID level selection %s."
msgstr ""
"Использование устройств типа %s требует определения надлежащего уровня RAID."

#: pyanaconda/ui/gui/spokes/custom.py:784
msgid "Device resize request failed. Click for details."
msgstr "Запрос на изменение размера устройства не выполнен. Подробнее…"

#: pyanaconda/ui/gui/spokes/custom.py:857
msgid "Device reformat request failed. Click for details."
msgstr "Запрос на переформатирование устройства не выполнен. Подробнее…"

#: pyanaconda/ui/gui/spokes/custom.py:1228
#: pyanaconda/ui/gui/spokes/custom.py:2114
#, fuzzy, python-format
msgid "Invalid device name: %s"
msgstr "Неверный часовой пояс"

#: pyanaconda/ui/gui/spokes/custom.py:1234
#, python-format
msgid "Specified name %s already in use."
msgstr "Имя %s уже используется."

#: pyanaconda/ui/gui/spokes/custom.py:1438
#, fuzzy
msgid "No disks assigned"
msgstr "Ни одного диска не выбрано"

#: pyanaconda/ui/gui/spokes/custom.py:1444
#, python-format
msgctxt "GUI|Custom Partitioning|Devices"
msgid " and %d other"
msgid_plural " and %d others"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: pyanaconda/ui/gui/spokes/custom.py:1502
#, fuzzy
msgid "The container is encrypted."
msgstr "%s не может быть зашифрован"

#: pyanaconda/ui/gui/spokes/custom.py:1538
msgid ""
"The space available to this mount point can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1540
msgid "This file system may not be resized."
msgstr "Для данной файловой системы не может быть изменен размер."

#: pyanaconda/ui/gui/spokes/custom.py:1594
#, fuzzy
msgid ""
"Error checking storage configuration.  Click for details or press Done again "
"to continue."
msgstr ""
"Ошибка при проверке конфигурации устройств хранения.  Нажмите для получения "
"подробных сведений."

#: pyanaconda/ui/gui/spokes/custom.py:1596
#, fuzzy
msgid ""
"Warning checking storage configuration.  Click for details or press Done "
"again to continue."
msgstr ""
"Ошибка при проверке конфигурации устройств хранения.  Нажмите для получения "
"подробных сведений."

#: pyanaconda/ui/gui/spokes/custom.py:1702
#, fuzzy, python-format
msgid "Added new %(type)s to existing container %(name)s."
msgstr "%s добавлен к существующему контейнеру %s."

#: pyanaconda/ui/gui/spokes/custom.py:1713
msgid "Failed to add new device. Click for details."
msgstr "Не удалось добавить новое устройство. Подробнее…"

#: pyanaconda/ui/gui/spokes/custom.py:1718
msgid "Invalid partition size set. Use a valid integer."
msgstr ""
"Недопустимый размер раздела. Необходимо указать корректное целое число."

#: pyanaconda/ui/gui/spokes/custom.py:1825
msgid "Device removal request failed. Click for details."
msgstr "Запрос на удаление устройства не выполнен. Подробнее…"

#: pyanaconda/ui/gui/spokes/custom.py:2055
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr "Группа томов %s уже используется. Изменения не будут сохранены."

#: pyanaconda/ui/gui/spokes/custom.py:2085
#, fuzzy, python-format
msgid "(%s free)"
msgstr "фс свободна"

#: pyanaconda/ui/gui/spokes/custom.py:2232
#, fuzzy, python-format
msgid ""
"This Software RAID array is missing %(missingMembers)d of %(totalMembers)d "
"member partitions. You can remove it or select a different device."
msgstr ""
"Это устройство не может быть изменено напрямую. Следует его удалить и "
"добавить другое устройство."

#: pyanaconda/ui/gui/spokes/custom.py:2238
#, fuzzy, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""
"Это устройство не может быть изменено напрямую. Следует его удалить и "
"добавить другое устройство."

#: pyanaconda/ui/gui/spokes/custom.py:2321
msgid "No disks selected."
msgstr "Не выбран ни один диск."

#: pyanaconda/ui/gui/spokes/custom.py:2326
msgid "Not enough free space on selected disks."
msgstr "Недостаточно свободного места на выбранных дисках."

#: pyanaconda/ui/gui/spokes/custom.py:2331
#: pyanaconda/ui/gui/spokes/custom.py:2356
msgid "Automatic partitioning failed. Click for details."
msgstr "Автоматическое разбиение не удалось. Подробнее…"

#: pyanaconda/ui/gui/spokes/custom.py:2580
msgid ""
"Continuing with this action will reset all your partitioning selections to "
"their current on-disk state."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2588
#, fuzzy
msgid "_Reset selections"
msgstr "_Изменить выбор программного обеспечения"

#: pyanaconda/ui/gui/spokes/custom.py:2588
msgid "_Preserve current selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2663
msgid "Failed to unlock encrypted block device. Click for details"
msgstr ""
"Не удалось расшифровать блочное устройство. Нажмите для получения подробных "
"сведений"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:401
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_TIME & DATE"
msgstr "ДАТА И ВРЕМЯ"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:524
msgid "Restoring hardware time..."
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:535
msgid "Invalid timezone"
msgstr "Неверный часовой пояс"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:711
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:725
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:911
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:751
msgid "PM"
msgstr "PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:713
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:722
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:910
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:913
msgid "AM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1061
msgid "You need to set up networking first if you want to use NTP"
msgstr "Для использования NTP нужно сначала настроить сеть"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1065
msgid "You have no working NTP server configured"
msgstr "Не настроено ещё не одного рабочего сервера NTP"

#: pyanaconda/ui/gui/spokes/filter.py:451
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_INSTALLATION DESTINATION"
msgstr "МЕСТО УСТАНОВКИ"

#: pyanaconda/ui/gui/spokes/filter.py:573
#, fuzzy, python-format
msgctxt "GUI|Installation Destination|Filter"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] "%d накопитель выбран"
msgstr[1] "%d накопителя выбрано"
msgstr[2] "%d накопителей выбрано"

#: pyanaconda/ui/gui/spokes/keyboard.py:48
#, python-format
msgid "%s to switch layouts."
msgstr "%s для переключения раскладок."

#: pyanaconda/ui/gui/spokes/keyboard.py:272
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_KEYBOARD"
msgstr "КЛАВИАТУРА"

#: pyanaconda/ui/gui/spokes/keyboard.py:359
msgid "Testing layouts configuration not available."
msgstr "Нельзя опробовать раскладки клавиатур."

#: pyanaconda/ui/gui/spokes/keyboard.py:367
#, fuzzy
msgid "Getting list of layouts..."
msgstr "проверка состояния..."

#: pyanaconda/ui/gui/spokes/keyboard.py:422
msgid "Layout switching not configured."
msgstr "Переключение раскладки не настроено."

#: pyanaconda/ui/gui/spokes/langsupport.py:54
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_LANGUAGE SUPPORT"
msgstr "ЯЗЫК"

#: pyanaconda/ui/gui/spokes/network.py:87
msgid "Status unknown (missing)"
msgstr "Статус неивестен (не найден)"

#: pyanaconda/ui/gui/spokes/network.py:90
msgid "Status unknown"
msgstr "Статус неивестен"

#: pyanaconda/ui/gui/spokes/network.py:92
msgid "Unmanaged"
msgstr "Неуправляемый"

#: pyanaconda/ui/gui/spokes/network.py:95
#: pyanaconda/ui/gui/spokes/network.py:99
msgid "Unavailable"
msgstr "Недоступно"

#: pyanaconda/ui/gui/spokes/network.py:97
msgid "Firmware missing"
msgstr "Прошивка не найдена"

#: pyanaconda/ui/gui/spokes/network.py:103
msgid "Cable unplugged"
msgstr "Кабель не подсоединён"

#: pyanaconda/ui/gui/spokes/network.py:105
msgid "Disconnected"
msgstr "Отключён"

#: pyanaconda/ui/gui/spokes/network.py:110
msgid "Connecting"
msgstr "Подключение"

#: pyanaconda/ui/gui/spokes/network.py:112
msgid "Authentication required"
msgstr "Требуется проверка подлинности"

#: pyanaconda/ui/gui/spokes/network.py:114
msgid "Connected"
msgstr "Подключен"

#: pyanaconda/ui/gui/spokes/network.py:116
msgid "Disconnecting"
msgstr "Отключение"

#: pyanaconda/ui/gui/spokes/network.py:118
msgid "Connection failed"
msgstr "Ошибка подключения"

#: pyanaconda/ui/gui/spokes/network.py:303
msgid "Ethernet"
msgstr "Ethernet"

#: pyanaconda/ui/gui/spokes/network.py:304
msgid "Wireless"
msgstr "Беспроводное"

#: pyanaconda/ui/gui/spokes/network.py:305
#: pyanaconda/ui/gui/spokes/network.glade:15
msgid "Bond"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:306
#: pyanaconda/ui/gui/spokes/network.glade:27
msgid "VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:307
#: pyanaconda/ui/gui/spokes/network.glade:23
msgid "Team"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:308
#: pyanaconda/ui/gui/spokes/network.glade:19
msgid "Bridge"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:729
msgid "unplugged"
msgstr "не подсоединён"

#: pyanaconda/ui/gui/spokes/network.py:819
#: pyanaconda/ui/gui/spokes/network.glade:540
#: pyanaconda/ui/gui/spokes/network.glade:841
msgid "IPv4 Address"
msgstr "Адрес IPv4:"

#: pyanaconda/ui/gui/spokes/network.py:820
#: pyanaconda/ui/gui/spokes/network.glade:516
#: pyanaconda/ui/gui/spokes/network.glade:853
#: pyanaconda/ui/gui/spokes/network.glade:1420
msgid "IPv6 Address"
msgstr "IPv6 Адрес"

#: pyanaconda/ui/gui/spokes/network.py:822
#: pyanaconda/ui/gui/spokes/network.py:824
#: pyanaconda/ui/gui/spokes/network.glade:1408
msgid "IP Address"
msgstr "IP-адрес"

#: pyanaconda/ui/gui/spokes/network.py:914
#, python-format
msgid "%d Mb/s"
msgstr "%d Мб/с"

#: pyanaconda/ui/gui/spokes/network.py:1134
msgid "WEP"
msgstr "WEP"

#: pyanaconda/ui/gui/spokes/network.py:1137
msgid "WPA"
msgstr "WPA"

#: pyanaconda/ui/gui/spokes/network.py:1140
msgid "WPA2"
msgstr "WPA2"

#: pyanaconda/ui/gui/spokes/network.py:1144
msgid "Enterprise"
msgstr "Enterprise"

#: pyanaconda/ui/gui/spokes/network.py:1149 widgets/src/SpokeSelector.c:59
#: pyanaconda/ui/gui/spokes/filter.glade:160
#: pyanaconda/ui/gui/spokes/filter.glade:566
#: pyanaconda/ui/gui/spokes/filter.glade:897
#: pyanaconda/ui/gui/spokes/filter.glade:1514
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:531
msgid "None"
msgstr "Нет"

#: pyanaconda/ui/gui/spokes/network.py:1281
#: pyanaconda/ui/gui/spokes/network.glade:2435
#, fuzzy
msgid "Authentication required by wireless network"
msgstr "Требуется проверка подлинности"

#: pyanaconda/ui/gui/spokes/network.py:1282
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1296
#, fuzzy
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Password:"
msgstr "Пароль:"

#: pyanaconda/ui/gui/spokes/network.py:1304
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Key:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1314
#, fuzzy
msgid "User name: "
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/network.py:1325
msgid "Identity: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1330
#, fuzzy
msgid "Private key password: "
msgstr "Пароль:"

#: pyanaconda/ui/gui/spokes/network.py:1386
msgctxt "GUI|Spoke"
msgid "_NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:52
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_ROOT PASSWORD"
msgstr "ПАРОЛЬ ROOT"

#: pyanaconda/ui/gui/spokes/password.py:99
#: pyanaconda/ui/gui/spokes/password.py:100
#, fuzzy
msgid "The password is set."
msgstr "Пароль установлен."

#: pyanaconda/ui/gui/spokes/password.py:127
msgid "Root password is set"
msgstr "Пароль root задан"

#: pyanaconda/ui/gui/spokes/password.py:129
msgid "Root account is disabled"
msgstr "Учетная запись пользователя root отключена"

#: pyanaconda/ui/gui/spokes/password.py:131
msgid "Root password is not set"
msgstr "Пароль root не установлен"

#: pyanaconda/ui/gui/spokes/software.py:52
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_SOFTWARE SELECTION"
msgstr "ВЫБОР ПРОГРАММНОГО ОБЕСПЕЧЕНИЯ"

#: pyanaconda/ui/gui/spokes/software.py:98
#: pyanaconda/ui/gui/spokes/source.py:54
msgid "Downloading package metadata..."
msgstr "Загрузка метаданных пакета..."

#: pyanaconda/ui/gui/spokes/software.py:101
msgid "Downloading group metadata..."
msgstr "Загрузка метаданных группы..."

#: pyanaconda/ui/gui/spokes/software.py:143
#: pyanaconda/ui/gui/spokes/source.py:529
msgid "Checking software dependencies..."
msgstr "Проверка зависимостей ПО..."

#: pyanaconda/ui/gui/spokes/software.py:148
msgid "Error checking software dependencies"
msgstr "Ошибка при проверке зависимостей"

#: pyanaconda/ui/gui/spokes/software.py:374
msgid "Error checking software dependencies.  Click for details."
msgstr ""
"Ошибка при проверке зависимостей ПО. Нажмите для получения подробных "
"сведений."

#: pyanaconda/ui/gui/spokes/software.py:478
#, fuzzy
msgid ""
"The software marked for installation has the following errors.  This is "
"likely caused by an error with your installation source.  You can quit the "
"installer, change your software source, or change your software selections."
msgstr ""
"Следующее программное обеспечение, отмеченное для установки, содержит "
"ошибки.  Вероятно, это вызвано ошибкой источника установки.  Можно "
"попытаться удалить эти пакеты из установки.\n"
"смените источник установки или выйдите из установщика."

#: pyanaconda/ui/gui/spokes/software.py:483
#, fuzzy
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Quit"
msgstr "В_ыход "

#: pyanaconda/ui/gui/spokes/software.py:484
#, fuzzy
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Modify Software Source"
msgstr "_Изменить источник ПО"

#: pyanaconda/ui/gui/spokes/software.py:485
#, fuzzy
msgctxt "GUI|Software Selection|Error Dialog"
msgid "Modify _Selections"
msgstr "_Изменить выбор программного обеспечения"

#: pyanaconda/ui/gui/spokes/source.py:53
msgid "Setting up installation source..."
msgstr "Настройка источника установки..."

#: pyanaconda/ui/gui/spokes/source.py:83
msgid "Invalid proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:88
#, python-format
msgid "Invalid proxy protocol: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:94
msgid "Extra characters in proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:98
#, fuzzy
msgid "Proxy authentication data duplicated"
msgstr "Требуется проверка подлинности"

#: pyanaconda/ui/gui/spokes/source.py:231
msgid "This media is good to install from."
msgstr "Этот носитель подходит для установки."

#: pyanaconda/ui/gui/spokes/source.py:233
msgid "This media is not good to install from."
msgstr "Этот носитель не подходит для установки."

#: pyanaconda/ui/gui/spokes/source.py:354
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_INSTALLATION SOURCE"
msgstr "ИСТОЧНИК УСТАНОВКИ"

#: pyanaconda/ui/gui/spokes/source.py:469
#: pyanaconda/ui/gui/spokes/source.py:673
#, fuzzy
msgid "Failed to set up installation source; check the repo url"
msgstr ""
"Не удалось настроить источник для установки, проверьте адрес репозитория"

#: pyanaconda/ui/gui/spokes/source.py:533
#, fuzzy
msgid "Error setting up base repository"
msgstr "Ошибка настройки репозитория"

#: pyanaconda/ui/gui/spokes/source.py:544
#, fuzzy
msgid "Error setting up ISO file"
msgstr "Ошибка при настройке источника программного обеспечения"

#: pyanaconda/ui/gui/spokes/source.py:675
#, fuzzy
msgid ""
"Failed to set up installation source; check the repo url and proxy settings"
msgstr ""
"Не удалось настроить источник для установки, проверьте адрес репозитория и "
"настройки прокси"

#: pyanaconda/ui/gui/spokes/source.py:704
#, fuzzy, python-format
msgid "Device: %s"
msgstr "Устройство:"

#: pyanaconda/ui/gui/spokes/source.py:705
#, python-format
msgid "Label: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:907
msgid "URL is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:911
msgid "Invalid URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:928
msgid "Protocol in URL does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:931
#, fuzzy
msgid "NFS server is empty"
msgstr "Сервер NFS %s"

#: pyanaconda/ui/gui/spokes/source.py:938
#, fuzzy
msgid "Invalid host name"
msgstr "Неверный часовой пояс"

#: pyanaconda/ui/gui/spokes/source.py:941
msgid "Remote directory is required"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:962
msgid "Duplicate repository names."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:969
#, fuzzy
msgid "Empty repository name"
msgstr "Неверный часовой пояс"

#: pyanaconda/ui/gui/spokes/source.py:972
#, fuzzy
msgid "Invalid repository name"
msgstr "Неверный часовой пояс"

#: pyanaconda/ui/gui/spokes/source.py:978
msgid "Repository name conflicts with internal repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:117
msgid "Please wait... software metadata still loading."
msgstr "Подождите… Всё ещё загружаются метаданные ПО."

#: pyanaconda/ui/gui/spokes/storage.py:120
#, fuzzy, python-format
msgid ""
"Your current <b>%(product)s</b> software selection requires <b>%(total)s</b> "
"of available space, including <b>%(software)s</b> for software and <b>"
"%(swap)s</b> for swap space."
msgstr ""
"Для текущего <b>%s</b> выбора программного обеспечения необходимо <b>%s</b> "
"свободного пространства."

#: pyanaconda/ui/gui/spokes/storage.py:129
#, python-format
msgid ""
"Your current <a href=\"\" title=\"%(tooltip)s\"><b>%(product)s</b> software "
"selection</a> requires <b>%(total)s</b> of available space, including <b>"
"%(software)s</b> for software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:179
#, fuzzy, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""
"%s\n"
"Выбранные диски имеют следующее количество свободного места:"

#: pyanaconda/ui/gui/spokes/storage.py:189
#, fuzzy, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""
"<b>Недостаточно свободного места для установки %s</b>, но можно\n"
"освободить место, сократив или удалив существующие разделы."

#: pyanaconda/ui/gui/spokes/storage.py:213
#, fuzzy, python-format
msgid ""
"  You don't have enough space available to install <b>%(product)s</b>, even "
"if you used all of the free space available on the selected disks."
msgstr ""
"%s\n"
"Не достаточно места для установки <b>%s</b>, даже если использовать всё "
"свободное пространство,\n"
"имеющееся на выбранных дисках."

#: pyanaconda/ui/gui/spokes/storage.py:225
#, fuzzy, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You "
"could add more disks for additional space, modify your software selection to "
"install a smaller version of <b>%(productName)s</b>, or quit the installer."
msgstr ""
"<b>Не достаточно места для установки %s</b>, даже если использовать всё "
"свободное пространство,\n"
"имеющееся на выбранных дисках.  Можно добавить диски для дополнительного "
"пространства,\n"
"изменить выбор программного обеспечения для получения меньшей по размеру "
"версии <b>%s</b>, или покинуть установщик."

#: pyanaconda/ui/gui/spokes/storage.py:246
#, fuzzy
msgctxt "GUI|Spoke"
msgid "INSTALLATION _DESTINATION"
msgstr "МЕСТО УСТАНОВКИ"

#: pyanaconda/ui/gui/spokes/storage.py:331
msgid "Saving storage configuration..."
msgstr "Сохранение конфигурации накопителей данных…"

#: pyanaconda/ui/gui/spokes/storage.py:337
#: pyanaconda/ui/gui/spokes/storage.py:349
msgid "Failed to save storage configuration..."
msgstr "Не удалось сохранить конфигурацию накопителей данных…"

#: pyanaconda/ui/gui/spokes/storage.py:385
#: pyanaconda/ui/gui/spokes/storage.py:682
#, fuzzy
msgid "Formatting DASDs"
msgstr "Форматирование DASD-устройства"

#: pyanaconda/ui/gui/spokes/storage.py:531
msgid "Error checking storage configuration.  Click for details."
msgstr ""
"Ошибка при проверке конфигурации устройств хранения.  Нажмите для получения "
"подробных сведений."

#: pyanaconda/ui/gui/spokes/storage.py:533
#, fuzzy
msgid "Warning checking storage configuration.  Click for details."
msgstr ""
"Ошибка при проверке конфигурации устройств хранения.  Нажмите для получения "
"подробных сведений."

#: pyanaconda/ui/gui/spokes/storage.py:574
#, python-format
msgid ""
"FCP device %(hba_id)s\n"
"WWPN %(wwpn)s\n"
"LUN %(lun)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:584
#, fuzzy, python-format
msgid "%s free"
msgstr "фс свободна"

#: pyanaconda/ui/gui/spokes/storage.py:631
#, fuzzy, python-format
msgid "%(count)d disk selected; %(capacity)s capacity; %(free)s free"
msgid_plural "%(count)d disks selected; %(capacity)s capacity; %(free)s free"
msgstr[0] "Выбран %d диск; емкость: %s; доступно: %s"
msgstr[1] "Выбрано %d диска; емкость: %s; доступно: %s"
msgstr[2] "Выбрано %d дисков; емкость: %s; доступно: %s"

#: pyanaconda/ui/gui/spokes/storage.py:715
#, fuzzy
msgid ""
"You have chosen to skip boot loader installation.  Your system may not be "
"bootable."
msgstr ""
"Вы решили пропустить установку загрузчика. Это может привести к тому, что "
"загрузка системы будет невозможна."

#: pyanaconda/ui/gui/spokes/storage.py:962
msgid "You'll be able to make space available during custom partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:978
#, fuzzy
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""
"Следующие ошибки были встречены при проверке конфигурации устройств "
"хранения.  Можете изменить схему устройств хранения\n"
"или выйти из программы установки."

#: pyanaconda/ui/gui/spokes/storage.py:983
#, fuzzy
msgctxt "GUI|Storage|Error Dialog"
msgid "_Quit"
msgstr "В_ыход "

#: pyanaconda/ui/gui/spokes/storage.py:984
#, fuzzy
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Storage Layout"
msgstr "_Изменить схему дисков"

#: pyanaconda/ui/gui/spokes/storage.py:998
#, fuzzy
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""
"Следующие ошибки были встречены при проверке конфигурации устройств "
"хранения.  Можете изменить схему устройств хранения\n"
"или выйти из программы установки."

#: pyanaconda/ui/gui/spokes/storage.py:1002
#, fuzzy
msgid "_OK"
msgstr "OК"

#: pyanaconda/ui/gui/spokes/user.py:72
#, python-format
msgid "Invalid group name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:215
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_USER CREATION"
msgstr "НАСТРОЙКА СЕТИ"

#: pyanaconda/ui/gui/spokes/user.py:293 pyanaconda/ui/gui/spokes/user.py:294
msgid "The password was set by kickstart."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:329
#, fuzzy
msgid "Invalid user name"
msgstr "Неверный часовой пояс"

#: pyanaconda/ui/gui/spokes/user.py:331
msgid "Full name cannot contain colon characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.py:258
#, fuzzy, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr "ДОБРО ПОЖАЛОВАТЬ В %s %s."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.py:269
#, python-format
msgid ""
"The following nodes were discovered using the iSCSI initiator <b>"
"%(initiatorName)s</b> using the target IP address <b>%(targetAddress)s</b>.  "
"Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:157
msgid "DATA"
msgstr "ДАННЫЕ"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:282
#, fuzzy, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr "Не создано ещё ни одной точки монтирования для установки %s %s:"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:291
#, fuzzy
msgid "_Click here to create them automatically."
msgstr "Нажмите для автоматического создания."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:316
#, fuzzy
msgid "Create new mount points by clicking the '+' button."
msgstr "Или же создайте новые точки монтирования ниже, нажав на значок '+' ."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:324
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:329
msgid "_New mount points will use the following partitioning scheme:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:131
#, fuzzy, python-format
msgid ""
"<b>%(count)d disk; %(size)s capacity; %(free)s free space</b> (unpartitioned "
"and in file systems)"
msgid_plural ""
"<b>%(count)d disks; %(size)s capacity; %(free)s free space</b> "
"(unpartitioned and in file systems)"
msgstr[0] ""
"<b>%d диск; емкость: %s; доступно: %s</b> (не занято разделами и файловыми "
"системами)"
msgstr[1] ""
"<b>%d диска; емкость: %s; доступно: %s</b> (не занято разделами и файловыми "
"системами)"
msgstr[2] ""
"<b>%d дисков; емкость: %s; доступно: %s</b> (не занято разделами и файловыми "
"системами)"

#: pyanaconda/ui/gui/spokes/lib/cart.py:190
msgctxt "GUI|Selected Disks Dialog"
msgid "_Do not install boot loader"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:193
#: pyanaconda/ui/gui/spokes/lib/cart.glade:183
#, fuzzy
msgctxt "GUI|Selected Disks Dialog"
msgid "_Set as Boot Device"
msgstr "_Назначить загрузочное устройство"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:64
#, fuzzy, python-format
msgid ""
"The RAID level you have selected (%(level)s) requires more disks (%(min)d) "
"than you currently have selected (%(count)d)."
msgstr ""
"Для выбранного уровня RAID требуется больше дисков, чем доступно на данный "
"момент."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:68
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:69
#, fuzzy, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr "Определите имя для группы томов и выберите как минимум один диск."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:75
#, fuzzy
msgid "Volume Group"
msgstr "Группа томов:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:75
#, fuzzy
msgid "_Volume Group:"
msgstr "Группа томов:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:76
#, fuzzy
msgid "Volume"
msgstr "колонка"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:76
#, fuzzy
msgid "_Volume:"
msgstr "колонка"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:136
#, fuzzy
msgid "Cannot relabel already existing file system."
msgstr "Точка монтирования %s должна располагаться в файловой системе Linux. "

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:141
#, fuzzy
msgid "Cannot set label on file system."
msgstr "Точка монтирования %s должна располагаться в файловой системе Linux. "

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:143
#, fuzzy
msgid "Unacceptable label format for file system."
msgstr "Точка монтирования %s должна располагаться в файловой системе Linux. "

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:153
msgid "That mount point is already in use. Try something else?"
msgstr "Эта точка монтирования уже используется. Попробовать ещё?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:155
#, fuzzy
msgid "Please enter a valid mount point."
msgstr "Укажите правильную точку монтирования."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:157
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:169
msgid "That mount point is invalid. Try something else?"
msgstr "Это неверная точка монтирования. Попробовать ещё?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:310
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:389
#, fuzzy, python-format
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Delete _all other file systems in the %s root as well."
msgstr "Удалить файловые системы в корне %s."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:399
#, python-format
msgid "Are you sure you want to delete all of the data on %s?"
msgstr "Удалить все данные с %s?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:401
#, fuzzy, python-format
msgid ""
"Are you sure you want to delete all of the data on %s, including snapshots "
"and/or subvolumes?"
msgstr "Удалить все данные с %s?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:685
#, fuzzy
msgid "Invalid container name"
msgstr "Неверный часовой пояс"

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:49
#, fuzzy
msgctxt "GUI|Detailed Error Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:59
#, fuzzy
msgctxt "GUI|Detailed Error Dialog"
msgid "_Quit"
msgstr "В_ыход "

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:35
#, python-format
msgid "You have provided a weak passphrase: %s"
msgstr "Указанный пароль слишком прост: %s"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:36
msgid "Passphrases do not match."
msgstr "Пароли не совпадают."

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:165
msgid "Passphrase contains non-ASCII characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:53
msgid "Preserve"
msgstr "Сохранить"

#: pyanaconda/ui/gui/spokes/lib/resize.py:54
msgid "Shrink"
msgstr "Ужать"

#: pyanaconda/ui/gui/spokes/lib/resize.py:55
msgid "Delete"
msgstr "Удалить"

#: pyanaconda/ui/gui/spokes/lib/resize.py:114
msgid "This device contains the installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:160
#, fuzzy, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr "Создание %(type)s на %(device)s"

#: pyanaconda/ui/gui/spokes/lib/resize.py:167
#, fuzzy
msgid "Not resizeable"
msgstr "Не готово"

#: pyanaconda/ui/gui/spokes/lib/resize.py:191
#, fuzzy
msgid "Free space"
msgstr "Использовать свободное пространство"

#: pyanaconda/ui/gui/spokes/lib/resize.py:213
#, fuzzy
msgid ""
"You can remove existing file systems you no longer need to free up space for "
"this installation.  Removing a file system will permanently delete all of "
"the data it contains."
msgstr ""
"Для этой установки не хватает свободного места.\n"
"\n"
"Чтобы освободить место, можно удалить ненужные файловые системы, но это "
"приведёт к полному удалению всех данных, которые в них находятся."

#: pyanaconda/ui/gui/spokes/lib/resize.py:219
msgid ""
"There is also free space available in pre-existing file systems.  While it's "
"risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:229
#, fuzzy, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in file systems)"
msgstr[0] "<b>%s диск; %s доступного места</b> (в файловых системах)"
msgstr[1] "<b>%s диска; %s доступного места</b> (в файловых системах)"
msgstr[2] "<b>%s дисков; %s доступного места</b> (в файловых системах)"

#: pyanaconda/ui/gui/spokes/lib/resize.py:236
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr "Всего выбрано места для высвобождения: <b>%s</b>"

#: pyanaconda/ui/gui/spokes/lib/resize.py:451
#: pyanaconda/ui/gui/spokes/lib/resize.py:456
#: pyanaconda/ui/gui/spokes/lib/resize.glade:269
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "Delete _all"
msgstr "Удалить"

#: pyanaconda/ui/gui/spokes/lib/resize.py:453
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "Preserve _all"
msgstr "Сохранить"

#: pyanaconda/ui/gui/spokes/lib/summary.py:54
#, python-format
msgid "%(description)s (%(deviceName)s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:58
#, python-format
msgid "%(deviceName)s on %(container)s"
msgstr ""

#: widgets/src/BaseWindow.c:111
msgid "DISTRIBUTION INSTALLATION"
msgstr "УСТАНОВКА ДИСТРИБУТИВА"

#: widgets/src/BaseWindow.c:112
msgid "SPOKE NAME"
msgstr "Название для запоминания"

#: widgets/src/BaseWindow.c:113
msgid "PRE-RELEASE / TESTING"
msgstr "ПРЕДВАРИТЕЛЬНЫЙ ВЫПУСК / ТЕСТИРУЕМАЯ ВЕРСИЯ"

#: widgets/src/DiskOverview.c:56
msgid "New Device"
msgstr "Новое устройство"

#: widgets/src/DiskOverview.c:58 widgets/src/DiskOverview.c:59
msgid "0 MB"
msgstr "0 Мб"

#: widgets/src/MountpointSelector.c:54
msgid "0 GB"
msgstr "0 Гб"

#: widgets/src/SpokeSelector.c:60
msgid "New Selector"
msgstr "Новый выбор"

#: widgets/src/SpokeWindow.c:48
msgid "_Done"
msgstr "Готово"

#: widgets/src/StandaloneWindow.c:52
msgid "_Continue"
msgstr "П_родолжить"

#: widgets/src/LayoutIndicator.c:34
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""

#: widgets/src/LayoutIndicator.c:35
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:36
#, fuzzy
msgctxt "GUI|Advanced User"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:50
#, fuzzy
msgctxt "GUI|Advanced User"
msgid "_Save Changes"
msgstr "_Применить"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:81
#, fuzzy
msgid "ADVANCED USER CONFIGURATION"
msgstr "НАСТРОЙКА СЕТИ"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:103
msgctxt "GUI|Advanced User"
msgid "Home _directory:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:138
msgid "User and Group IDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:157
msgctxt "GUI|Advanced User"
msgid "Specify a _user ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:173
msgctxt "GUI|Advanced User"
msgid "Specify a _group ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:231
#, fuzzy
msgid "Group Membership"
msgstr "Имя группы"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:259
msgctxt "GUI|Advanced User"
msgid "_Add user to the following groups:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:290
msgid "Example:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:307
msgid "wheel, my-team (1245), project-x (29935)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:341
msgid "Tip:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:360
msgid ""
"You may input a comma-separated list of group names and group IDs here. "
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:110
#, fuzzy
msgctxt "GUI|Date and Time|NTP"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:124
#, fuzzy
msgctxt "GUI|Date and Time|NTP"
msgid "_OK"
msgstr "OК"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:149
msgid "Add and mark for usage NTP servers"
msgstr "Добавить и отметить для использования NTP серверы"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:171
#, fuzzy
msgid "New NTP Server"
msgstr "Веб-сервер"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:190
msgid "Add NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:210
msgid "This URL refers to a pool of NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:256
msgid "Pool"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:269
msgid "Working"
msgstr "В процессе"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:277
msgid "Use"
msgstr "Использовать"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:305
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:519
#, fuzzy
msgid "Configure NTP"
msgstr "_Настроить..."

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:326
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:934
#, fuzzy
msgid "TIME & DATE"
msgstr "ДАТА И ВРЕМЯ"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:376
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_Region:"
msgstr "Регион:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:406
#, fuzzy
msgid "Region"
msgstr "Регион:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:423
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_City:"
msgstr "Город:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:453
#, fuzzy
msgid "City"
msgstr "Город:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:481
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_Network Time"
msgstr "_Сетевое имя"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:499
#, fuzzy
msgid "Use Network Time"
msgstr "Сетевое имя"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:586
msgid "Hours"
msgstr ""

#. TRANSLATORS: This is the separator between hours and minutes, like in HH:MM
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:599
msgid ":"
msgstr ":"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:619
msgid "Minutes"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:637
msgid "Hour Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:655
msgid "Hour Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:673
msgid "Minutes Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:691
msgid "Minutes Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:735
#, fuzzy
msgid "AM/PM Up"
msgstr "AM/PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:771
#, fuzzy
msgid "AM/PM Down"
msgstr "AM/PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:822
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "24-_hour"
msgstr "24 часа"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:840
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_AM/PM"
msgstr "AM/PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:899
msgid "Set Date & Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:60
#: pyanaconda/ui/gui/spokes/storage.glade:444
#: pyanaconda/ui/gui/spokes/storage.glade:1024
msgid "INSTALLATION DESTINATION"
msgstr "МЕСТО УСТАНОВКИ"

#: pyanaconda/ui/gui/spokes/filter.glade:133
msgid "Search Results:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:146
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:161
msgid "Port / Target / LUN #"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:162
msgid "Target WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:199
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Port:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:224
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Target:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:252
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:293
msgctxt "GUI|Installation Destination|Filter|Search|WWID"
msgid "_WWID:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:366
#: pyanaconda/ui/gui/spokes/filter.glade:1267
#: pyanaconda/ui/gui/spokes/keyboard.glade:520
#: pyanaconda/ui/gui/spokes/source.glade:987
#: pyanaconda/ui/gui/spokes/lib/cart.glade:133
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:235
#: pyanaconda/ui/gui/spokes/lib/resize.glade:163
msgid "Name"
msgstr "Имя"

#: pyanaconda/ui/gui/spokes/filter.glade:380
#: pyanaconda/ui/gui/spokes/filter.glade:569
#: pyanaconda/ui/gui/spokes/filter.glade:748
msgid "WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:394
#: pyanaconda/ui/gui/spokes/filter.glade:762
#: pyanaconda/ui/gui/spokes/filter.glade:954
#: pyanaconda/ui/gui/spokes/filter.glade:1295
#: pyanaconda/ui/gui/spokes/lib/cart.glade:145
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:248
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:726
msgid "Capacity"
msgstr "Объем"

#: pyanaconda/ui/gui/spokes/filter.glade:408
#: pyanaconda/ui/gui/spokes/filter.glade:568
#: pyanaconda/ui/gui/spokes/filter.glade:791
#: pyanaconda/ui/gui/spokes/filter.glade:899
#: pyanaconda/ui/gui/spokes/filter.glade:983
#, fuzzy
msgid "Interconnect"
msgstr "Не подключен"

#: pyanaconda/ui/gui/spokes/filter.glade:422
msgid "Model"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:436
#: pyanaconda/ui/gui/spokes/filter.glade:1323
#: pyanaconda/ui/gui/spokes/filter.glade:1517
msgid "LUN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:450
#: pyanaconda/ui/gui/spokes/filter.glade:1337
msgid "Port"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:464
msgid "Target"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:478
#: pyanaconda/ui/gui/spokes/filter.glade:567
#: pyanaconda/ui/gui/spokes/filter.glade:776
#: pyanaconda/ui/gui/spokes/filter.glade:898
#: pyanaconda/ui/gui/spokes/filter.glade:968
msgid "Vendor"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:499
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:521
msgctxt "GUI|Installation Destination|Filter"
msgid "Searc_h"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:552
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:605
msgctxt "GUI|Installation Destination|Filter|Multipath|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:643
msgctxt "GUI|Installation Destination|Filter|Multipath|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:681
msgctxt "GUI|Installation Destination|Filter|Multipath|WWID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:805
msgid "Paths"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:826
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:851
msgctxt "GUI|Installation Destination|Filter"
msgid "_Multipath Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:883
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:900
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:752
msgid "ID"
msgstr "ID"

#: pyanaconda/ui/gui/spokes/filter.glade:940
msgid "Identifier"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1046
msgctxt "GUI|Installation Destination|Filter|Other|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1084
msgctxt "GUI|Installation Destination|Filter|Other|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1122
msgctxt "GUI|Installation Destination|Filter|Other|ID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1173
#, fuzzy
msgctxt "GUI|Installation Destination|Filter"
msgid "_Other SAN Devices"
msgstr "Выбранное устройство"

#: pyanaconda/ui/gui/spokes/filter.glade:1205
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1215
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1253
#: pyanaconda/ui/gui/spokes/filter.glade:1515
msgid "CCW"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1281
#: pyanaconda/ui/gui/spokes/lib/summary.glade:155
#, fuzzy
msgid "Type"
msgstr "Тип VPN"

#: pyanaconda/ui/gui/spokes/filter.glade:1309
#: pyanaconda/ui/gui/spokes/filter.glade:1516
msgid "WWPN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1382
msgctxt "GUI|Installation Destination|Filter|zSeries|CCW"
msgid "_CCW:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1429
msgctxt "GUI|Installation Destination|Filter|zSeries|WWPN"
msgid "_WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1471
msgctxt "GUI|Installation Destination|Filter|zSeries|LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1541
#, fuzzy
msgctxt "GUI|Installation Destination|Filter"
msgid "_zSeries Devices"
msgstr "Выбранное устройство"

#: pyanaconda/ui/gui/spokes/filter.glade:1564
msgctxt "GUI|Installation Destination|Filter"
msgid "_Add zFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1579
msgctxt "GUI|Installation Destination|Filter"
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1594
msgctxt "GUI|Installation Destination|Filter"
msgid "Add _iSCSI Target..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1609
msgctxt "GUI|Installation Destination|Filter"
msgid "Add FCo_E SAN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1624
msgid "Refresh _List"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:35
#, fuzzy
msgid "Add device"
msgstr "Устройство RAID"

#: pyanaconda/ui/gui/spokes/network.glade:49
#, fuzzy
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/network.glade:63
#, fuzzy
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Add"
msgstr "_Добавить"

#: pyanaconda/ui/gui/spokes/network.glade:92
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Select the type of device you wish to add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:165
#: pyanaconda/ui/gui/spokes/network.glade:2270
msgid "NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:204
msgid ""
"Please use the live desktop environment's tools for customizing your network "
"configuration.  You can set the host name here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:418
msgid "Slaves"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:442
#: pyanaconda/ui/gui/spokes/network.glade:878
#: pyanaconda/ui/gui/spokes/network.glade:1445
msgid "DNS"
msgstr "DNS"

#: pyanaconda/ui/gui/spokes/network.glade:468
#: pyanaconda/ui/gui/spokes/network.glade:865
#: pyanaconda/ui/gui/spokes/network.glade:1432
msgid "Default Route"
msgstr "Маршрут по умолчанию"

#: pyanaconda/ui/gui/spokes/network.glade:492
msgid "Subnet Mask"
msgstr "Маска подсети"

#: pyanaconda/ui/gui/spokes/network.glade:564
#: pyanaconda/ui/gui/spokes/network.glade:829
#: pyanaconda/ui/gui/spokes/network.glade:1276
msgid "Speed"
msgstr "Скорость"

#: pyanaconda/ui/gui/spokes/network.glade:588
#: pyanaconda/ui/gui/spokes/network.glade:817
msgid "Hardware Address"
msgstr "Аппаратный адрес"

#: pyanaconda/ui/gui/spokes/network.glade:612
msgid "Parent"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:624
msgid "VLAN ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:699
#, fuzzy
msgctxt "GUI|Network|Wired"
msgid "_Configure..."
msgstr "_Настроить..."

#: pyanaconda/ui/gui/spokes/network.glade:703
msgid "To apply the configuration immediately turn the device off and on."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:990
msgid "Security"
msgstr "Безопасность"

#: pyanaconda/ui/gui/spokes/network.glade:1002
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Network Name"
msgstr "_Сетевое имя"

#: pyanaconda/ui/gui/spokes/network.glade:1016
msgid "Network Name"
msgstr "Сетевое имя"

#: pyanaconda/ui/gui/spokes/network.glade:1028
msgid "Security Key"
msgstr "Ключ безопасности"

#: pyanaconda/ui/gui/spokes/network.glade:1111
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Use as Hotspot..."
msgstr "_Использовать как точку доступа..."

#: pyanaconda/ui/gui/spokes/network.glade:1128
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Stop Hotspot..."
msgstr "_Остановить точку доступа..."

#: pyanaconda/ui/gui/spokes/network.glade:1145
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Configure..."
msgstr "_Настроить..."

#: pyanaconda/ui/gui/spokes/network.glade:1264
msgid "IMEI"
msgstr "IMEI"

#: pyanaconda/ui/gui/spokes/network.glade:1288
msgid "Provider"
msgstr "Поставщик услуг"

#: pyanaconda/ui/gui/spokes/network.glade:1470
#, fuzzy
msgctxt "GUI|Network|Mobile Broadband"
msgid "_Configure..."
msgstr "_Настроить..."

#: pyanaconda/ui/gui/spokes/network.glade:1581
msgid "VPN Type"
msgstr "Тип VPN"

#: pyanaconda/ui/gui/spokes/network.glade:1593
msgid "Gateway"
msgstr "Шлюз"

#: pyanaconda/ui/gui/spokes/network.glade:1605
msgid "Group Name"
msgstr "Имя группы"

#: pyanaconda/ui/gui/spokes/network.glade:1617
msgid "Group Password"
msgstr "Пароль группы"

#: pyanaconda/ui/gui/spokes/network.glade:1629
#: pyanaconda/ui/gui/spokes/user.glade:116
#, fuzzy
msgid "User Name"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/network.glade:1736
#, fuzzy
msgctxt "GUI|Network|VPN"
msgid "_Configure..."
msgstr "_Настроить..."

#: pyanaconda/ui/gui/spokes/network.glade:1859
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "_Method"
msgstr "_Метод"

#: pyanaconda/ui/gui/spokes/network.glade:1884
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "_Configuration URL"
msgstr "_Настройки адреса"

#: pyanaconda/ui/gui/spokes/network.glade:1909
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "HTTP _Proxy"
msgstr "_HTTP Proxy"

#: pyanaconda/ui/gui/spokes/network.glade:1923
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "H_TTPS Proxy"
msgstr "H_TTPS Proxy"

#: pyanaconda/ui/gui/spokes/network.glade:1937
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "_FTP Proxy"
msgstr "_FTP Proxy"

#: pyanaconda/ui/gui/spokes/network.glade:1951
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "_Socks Host"
msgstr "_Узел socks"

#: pyanaconda/ui/gui/spokes/network.glade:2140
#, fuzzy
msgid "Network Config Box"
msgstr "Проверка настроек накопителей..."

#: pyanaconda/ui/gui/spokes/network.glade:2156
#, fuzzy
msgctxt "GUI|Network"
msgid "Unloc_k"
msgstr "Разблокировать"

#: pyanaconda/ui/gui/spokes/network.glade:2177
#, fuzzy
msgctxt "GUI|Network"
msgid "_Airplane Mode"
msgstr "Ре_жим полёта"

#: pyanaconda/ui/gui/spokes/network.glade:2196
#, fuzzy
msgid "More Network Config Box"
msgstr "Проверка настроек накопителей..."

#: pyanaconda/ui/gui/spokes/network.glade:2216
#, fuzzy
msgctxt "GUI|Network"
msgid "_Host Name:"
msgstr "Имя сервера"

#: pyanaconda/ui/gui/spokes/network.glade:2276
msgid "NETWORK CONFIGURATION"
msgstr "НАСТРОЙКА СЕТИ"

#: pyanaconda/ui/gui/spokes/network.glade:2321
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""
"Необходим доступ к сети, что бы получать информацию о местонахождении и "
"сделать\n"
"доступными обновления для программного обеспечения."

#: pyanaconda/ui/gui/spokes/network.glade:2351
#, fuzzy
msgid "Authentication"
msgstr "_Использовать проверку подлинности"

#: pyanaconda/ui/gui/spokes/network.glade:2365
#, fuzzy
msgctxt "GUI|Network|Authentication Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/network.glade:2380
#, fuzzy
msgctxt "GUI|Network|Authentication Dialog"
msgid "C_onnect"
msgstr "Подключен"

#: pyanaconda/ui/gui/spokes/network.glade:2454
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:10
msgid "SOFTWARE SELECTION"
msgstr "ВЫБОР ПРОГРАММНОГО ОБЕСПЕЧЕНИЯ"

#: pyanaconda/ui/gui/spokes/software.glade:66
#, fuzzy
msgid "Base Environment"
msgstr "Выберите среду"

#: pyanaconda/ui/gui/spokes/software.glade:84
msgid "Add-Ons for Selected Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:27
#, fuzzy
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Cancel & add more disks"
msgstr "Отменить и _добавить больше дисков"

#: pyanaconda/ui/gui/spokes/storage.glade:43
#, fuzzy
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Reclaim space"
msgstr "_Вернуть пространство"

#: pyanaconda/ui/gui/spokes/storage.glade:78
#: pyanaconda/ui/gui/spokes/storage.glade:293
msgid "INSTALLATION OPTIONS"
msgstr "ПАРАМЕТРЫ УСТАНОВКИ"

#: pyanaconda/ui/gui/spokes/storage.glade:95
#: pyanaconda/ui/gui/spokes/storage.glade:309
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""
"Здесь описывается, сколько места нужно для текущего выбора программного "
"обеспечения."

#: pyanaconda/ui/gui/spokes/storage.glade:121
#: pyanaconda/ui/gui/spokes/storage.glade:335
msgid "disk free"
msgstr "диск чист"

#: pyanaconda/ui/gui/spokes/storage.glade:137
#: pyanaconda/ui/gui/spokes/storage.glade:350
msgid "Free space available for use."
msgstr "Свободное пространство, доступное для использования."

#: pyanaconda/ui/gui/spokes/storage.glade:153
#: pyanaconda/ui/gui/spokes/storage.glade:366
msgid "fs free"
msgstr "фс свободна"

#: pyanaconda/ui/gui/spokes/storage.glade:169
#: pyanaconda/ui/gui/spokes/storage.glade:382
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr "Свободного места нет, но его можно получить из существующих разделов."

#: pyanaconda/ui/gui/spokes/storage.glade:192
#: pyanaconda/ui/gui/spokes/storage.glade:405
msgid "Here we'll describe what your options are."
msgstr "Здесь описываются варианты выбора."

#: pyanaconda/ui/gui/spokes/storage.glade:219
#, fuzzy
msgid "Need Space"
msgstr "Использовать свободное пространство"

#: pyanaconda/ui/gui/spokes/storage.glade:242
#, fuzzy
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Quit installer"
msgstr "_Выйти из установщика"

#: pyanaconda/ui/gui/spokes/storage.glade:258
#, fuzzy
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/storage.glade:432
msgid "No Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:507
#, fuzzy
msgid "Device Selection"
msgstr "Описание устройства"

#: pyanaconda/ui/gui/spokes/storage.glade:526
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:544
msgid "Local Standard Disks"
msgstr "Локальные стандартные диски"

#: pyanaconda/ui/gui/spokes/storage.glade:608
#: pyanaconda/ui/gui/spokes/storage.glade:748
msgid "Disks left unselected here will not be touched."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:627
msgid "Specialized & Network Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:697
msgctxt "GUI|Storage"
msgid "_Add a disk..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:714
msgid "Add Specialized Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:767
msgid "Other Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:793
#, fuzzy
msgid "Partitioning"
msgstr "_Завершить разбиение"

#: pyanaconda/ui/gui/spokes/storage.glade:810
#, fuzzy
msgid "Encryption"
msgstr "Зашифровать"

#: pyanaconda/ui/gui/spokes/storage.glade:823
#, fuzzy
msgid "A_utomatically configure partitioning."
msgstr "Выбрано автоматическое разбиение диска"

#: pyanaconda/ui/gui/spokes/storage.glade:839
#, fuzzy
msgid "_I will configure partitioning."
msgstr "_Завершить разбиение"

#: pyanaconda/ui/gui/spokes/storage.glade:871
#, fuzzy
msgid "_Encrypt my data."
msgstr "Зашифровать"

#: pyanaconda/ui/gui/spokes/storage.glade:886
#, fuzzy
msgid "You'll set a passphrase next."
msgstr "Шифровать данные. Пароль будет установлен позднее."

#: pyanaconda/ui/gui/spokes/storage.glade:907
msgid "I would like to _make additional space available."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:923
#, fuzzy
msgid "Storage Options"
msgstr "Настройки автоматического создания разделов"

#: pyanaconda/ui/gui/spokes/storage.glade:955
msgid "summary"
msgstr "сводка"

#: pyanaconda/ui/gui/spokes/storage.glade:988
#, fuzzy
msgctxt "GUI|Storage"
msgid "_Full disk summary and boot loader..."
msgstr "Полная сводка данных о дисках и опциях…"

#: pyanaconda/ui/gui/spokes/keyboard.glade:15
#: pyanaconda/ui/gui/spokes/keyboard.glade:388
msgid "KEYBOARD LAYOUT"
msgstr "РАСКЛАДКА КЛАВИАТУРЫ"

#: pyanaconda/ui/gui/spokes/keyboard.glade:57
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""
"Какую раскладку клавиатуры использовать на этой системе? Раскладка, "
"передвинутая на верх списка, будет использоваться по умолчанию."

#: pyanaconda/ui/gui/spokes/keyboard.glade:96
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:153
msgid "name"
msgstr "имя"

#: pyanaconda/ui/gui/spokes/keyboard.glade:167
#, fuzzy
msgid "Selected Layouts"
msgstr "Передвинуть выбранную раскладку вверх"

#: pyanaconda/ui/gui/spokes/keyboard.glade:190
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Add layout"
msgstr "Добавить раскладку"

#: pyanaconda/ui/gui/spokes/keyboard.glade:205
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Remove layout"
msgstr "Удалить раскладку"

#: pyanaconda/ui/gui/spokes/keyboard.glade:220
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _up"
msgstr "Передвинуть выбранную раскладку вверх"

#: pyanaconda/ui/gui/spokes/keyboard.glade:235
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _down"
msgstr "Передвинуть выбранную раскладку вниз"

#: pyanaconda/ui/gui/spokes/keyboard.glade:251
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Preview layout"
msgstr "Посмотреть раскладку"

#: pyanaconda/ui/gui/spokes/keyboard.glade:291
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Test the layout configuration below:"
msgstr "Нельзя опробовать раскладки клавиатур."

#: pyanaconda/ui/gui/spokes/keyboard.glade:331
msgid "Alt + Shift to switch layouts."
msgstr "Alt + Shift для переключения раскладок."

#: pyanaconda/ui/gui/spokes/keyboard.glade:346
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Options"
msgstr "_Параметры"

#: pyanaconda/ui/gui/spokes/keyboard.glade:421
#, fuzzy
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/keyboard.glade:435
#, fuzzy
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Add"
msgstr "_Добавить"

#: pyanaconda/ui/gui/spokes/keyboard.glade:467
msgid "ADD A KEYBOARD LAYOUT"
msgstr "ДОБАВИТЬ РАСКЛАДКУ КЛАВИАТУРЫ"

#: pyanaconda/ui/gui/spokes/keyboard.glade:484
msgid "You may add a keyboard layout by selecting it below:"
msgstr "Можно добавить раскладку клавиатуры, выбрав её ниже:"

#: pyanaconda/ui/gui/spokes/keyboard.glade:533
#, fuzzy
msgid "Available Layouts"
msgstr "Доступные регионы"

#: pyanaconda/ui/gui/spokes/keyboard.glade:574
#, fuzzy
msgid "Add Layout"
msgstr "Добавить раскладку"

#: pyanaconda/ui/gui/spokes/keyboard.glade:606
#, fuzzy
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/keyboard.glade:620
#, fuzzy
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_OK"
msgstr "OК"

#: pyanaconda/ui/gui/spokes/keyboard.glade:652
msgid "LAYOUT SWITCHING OPTIONS"
msgstr "ПАРАМЕТРЫ ПЕРЕКЛЮЧЕНИЯ РАСКЛАДКИ"

#: pyanaconda/ui/gui/spokes/keyboard.glade:669
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr "Выберите комбинацию клавиш для переключения раскладки."

#: pyanaconda/ui/gui/spokes/keyboard.glade:705
msgid "use"
msgstr "использование"

#: pyanaconda/ui/gui/spokes/keyboard.glade:718
msgid "description"
msgstr "описание"

#: pyanaconda/ui/gui/spokes/keyboard.glade:752
#, fuzzy
msgid "Layout Options"
msgstr "Параметры монтирования NFS:"

#: pyanaconda/ui/gui/spokes/langsupport.glade:33
#, fuzzy
msgid "LANGUAGE SUPPORT"
msgstr "ЯЗЫК"

#: pyanaconda/ui/gui/spokes/langsupport.glade:79
msgid "Select additional language support to be installed:"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:138
#: pyanaconda/ui/gui/spokes/langsupport.glade:241
#: pyanaconda/ui/gui/spokes/welcome.glade:273
#: pyanaconda/ui/gui/spokes/welcome.glade:372
msgid "nativeName"
msgstr "родноеНазвание"

#: pyanaconda/ui/gui/spokes/langsupport.glade:155
#: pyanaconda/ui/gui/spokes/welcome.glade:290
msgid "englishName"
msgstr "английскоеНазвание"

#: pyanaconda/ui/gui/spokes/langsupport.glade:173
#: pyanaconda/ui/gui/spokes/welcome.glade:308
#, fuzzy
msgid "selected"
msgstr "Ничего не выбрано"

#: pyanaconda/ui/gui/spokes/langsupport.glade:193
#: pyanaconda/ui/gui/spokes/welcome.glade:336
msgid "Type here to search."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:227
msgid "checked"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:8
#: pyanaconda/ui/gui/spokes/password.glade:203
msgid "ROOT PASSWORD"
msgstr "ПАРОЛЬ ROOT"

#: pyanaconda/ui/gui/spokes/password.glade:54
#, fuzzy
msgctxt "GUI|Password"
msgid "_Root Password:"
msgstr "Пароль root:"

#: pyanaconda/ui/gui/spokes/password.glade:68
#, fuzzy
msgctxt "GUI|Password"
msgid "_Confirm:"
msgstr "Подтверждение:"

#: pyanaconda/ui/gui/spokes/password.glade:104
#: pyanaconda/ui/gui/spokes/user.glade:187
#, fuzzy
msgid "Confirm Password"
msgstr "Пароль VNC"

#: pyanaconda/ui/gui/spokes/password.glade:150
#: pyanaconda/ui/gui/spokes/user.glade:251
#, fuzzy
msgid "empty password"
msgstr "Установите пароль root"

#: pyanaconda/ui/gui/spokes/password.glade:171
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"Учетная запись администратора (root) предназначена для управления системой. "
"Введите пароль root."

#: pyanaconda/ui/gui/spokes/source.glade:29
#, fuzzy
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/source.glade:43
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Open"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:95
#, fuzzy
msgctxt "GUI|Software Source|Media Check Dialog"
msgid "_Done"
msgstr "Готово"

#: pyanaconda/ui/gui/spokes/source.glade:128
msgid "MEDIA VERIFICATION"
msgstr "ПРОВЕРКА НОСИТЕЛЕЙ"

#: pyanaconda/ui/gui/spokes/source.glade:155
msgid "Verifying media, please wait..."
msgstr "Проверка носителей, подождите..."

#: pyanaconda/ui/gui/spokes/source.glade:224
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/source.glade:239
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_OK"
msgstr "OК"

#: pyanaconda/ui/gui/spokes/source.glade:267
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Enable HTTP Proxy"
msgstr "_Разрешить HTTP Proxy"

#: pyanaconda/ui/gui/spokes/source.glade:300
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Proxy URL:"
msgstr "<b>Адрес прокси</b>"

#: pyanaconda/ui/gui/spokes/source.glade:317
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr "<span size=\"small\"><b>Пример:</b> squid.mysite.org:3128</span>"

#: pyanaconda/ui/gui/spokes/source.glade:348
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Use Authentication"
msgstr "_Использовать проверку подлинности"

#: pyanaconda/ui/gui/spokes/source.glade:375
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "User _name:"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/source.glade:392
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "Pass_word:"
msgstr "Пароль:"

#: pyanaconda/ui/gui/spokes/source.glade:471
msgid "INSTALLATION SOURCE"
msgstr "ИСТОЧНИК УСТАНОВКИ"

#: pyanaconda/ui/gui/spokes/source.glade:530
msgid "Which installation source would you like to use?"
msgstr "Выберите источник установки"

#: pyanaconda/ui/gui/spokes/source.glade:544
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Auto-detected installation media:"
msgstr "_Автоматически обнаруженный установочный носитель:"

#: pyanaconda/ui/gui/spokes/source.glade:571
msgid "Device:"
msgstr "Устройство:"

#: pyanaconda/ui/gui/spokes/source.glade:583
#, fuzzy
msgid "Label:"
msgstr "<b>Метка:</b>"

#: pyanaconda/ui/gui/spokes/source.glade:592
#: pyanaconda/ui/gui/spokes/source.glade:692
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Verify"
msgstr "_Проверить"

#: pyanaconda/ui/gui/spokes/source.glade:616
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_ISO file:"
msgstr "_ISO-файл:"

#: pyanaconda/ui/gui/spokes/source.glade:644
#, fuzzy
msgctxt "GUI|Software Source"
msgid "D_evice:"
msgstr "Устройство:"

#: pyanaconda/ui/gui/spokes/source.glade:675
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Choose an ISO"
msgstr "_Выбрать ISO"

#: pyanaconda/ui/gui/spokes/source.glade:716
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_On the network:"
msgstr "По _сети"

#: pyanaconda/ui/gui/spokes/source.glade:779
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Proxy setup..."
msgstr "_Настройки прокси..."

#: pyanaconda/ui/gui/spokes/source.glade:794
msgctxt "GUI|Software Source"
msgid "This URL refers to a _mirror list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:832
#, fuzzy
msgctxt "GUI|Software Source"
msgid "N_FS mount options:"
msgstr "Параметры монтирования NFS:"

#: pyanaconda/ui/gui/spokes/source.glade:847
#: pyanaconda/ui/gui/spokes/source.glade:848
msgid "This field is optional."
msgstr "Это поле не является обязательным."

#: pyanaconda/ui/gui/spokes/source.glade:881
msgid "Updates"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:908
msgctxt "GUI|Software Source"
msgid ""
"Don't install the latest available software _updates. Install the default "
"versions provided by the installation source above."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:933
#, fuzzy
msgid "Additional repositories"
msgstr "Репозитории Fedora People"

#: pyanaconda/ui/gui/spokes/source.glade:974
msgid "Enabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1019
#: pyanaconda/ui/gui/spokes/source.glade:1020
#, fuzzy
msgid "Add a new repository."
msgstr "Добавить точку монтирования"

#: pyanaconda/ui/gui/spokes/source.glade:1021
msgctxt "GUI|Software Source"
msgid "A_dd"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1038
#: pyanaconda/ui/gui/spokes/source.glade:1039
#, fuzzy
msgid "Remove the selected repository."
msgstr "Удалить выбранные точки монтирования."

#: pyanaconda/ui/gui/spokes/source.glade:1040
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Remove"
msgstr "Удалить раскладку"

#: pyanaconda/ui/gui/spokes/source.glade:1057
#: pyanaconda/ui/gui/spokes/source.glade:1058
msgid "Revert to the previous list of repositories."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1059
msgctxt "GUI|Software Source"
msgid "Rese_t"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1089
#: pyanaconda/ui/gui/spokes/source.glade:1090
msgid "URL for the repository, without protocol."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1102
#: pyanaconda/ui/gui/spokes/source.glade:1103
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1115
#: pyanaconda/ui/gui/spokes/source.glade:1116
#, fuzzy
msgid "Optional proxy user name."
msgstr "Операционная ошибка."

#: pyanaconda/ui/gui/spokes/source.glade:1128
#, fuzzy
msgid "Optional proxy password."
msgstr "Установите пароль root"

#: pyanaconda/ui/gui/spokes/source.glade:1140
msgctxt "GUI|Software Source"
msgid "This URL refers to a mirror _list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1160
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Name:"
msgstr "Имя:"

#: pyanaconda/ui/gui/spokes/source.glade:1176
#: pyanaconda/ui/gui/spokes/source.glade:1177
msgid "Protocol for the repository URL."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1198
#, fuzzy
msgctxt "GUI|Software Source"
msgid "Pro_xy URL:"
msgstr "<b>Адрес прокси</b>"

#: pyanaconda/ui/gui/spokes/source.glade:1215
#, fuzzy
msgctxt "GUI|Software Source"
msgid "U_ser name:"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/source.glade:1232
#, fuzzy
msgctxt "GUI|Software Source"
msgid "Pass_word:"
msgstr "Пароль:"

#: pyanaconda/ui/gui/spokes/source.glade:1248
#: pyanaconda/ui/gui/spokes/source.glade:1249
msgid "Name of the repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:23
#, fuzzy
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_Get me out of here!"
msgstr "_Уйти отсюда!"

#: pyanaconda/ui/gui/spokes/welcome.glade:37
#, fuzzy
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_I accept my fate."
msgstr "_Продолжить"

#: pyanaconda/ui/gui/spokes/welcome.glade:68
msgid "This is unstable, pre-release software."
msgstr "Стабильность предварительной версии дистрибутива не гарантируется"

#: pyanaconda/ui/gui/spokes/welcome.glade:97
msgid ""
"You wake up inside an OS installer in Timbuktu, and it's six months in the "
"future.  But, there are bugs.  Bugs everywhere.  Bugs you must live with.  "
"This OS of the future isn't a stable OS you can rely on.  It's for testing "
"purposes only."
msgstr ""
"Вы оказались в установщике непонятной ОС из будущего. Пока в ней множество "
"ошибок, с которыми придется мириться. Эта система не является той, на "
"которую можно положиться. Пока она предназначена только для тестирования."

#: pyanaconda/ui/gui/spokes/welcome.glade:126
msgid "Beta Warn"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:225
msgid "What language would you like to use during the installation process?"
msgstr "Какой язык вы хотите использовать в процессе установки?"

#: pyanaconda/ui/gui/spokes/welcome.glade:316
msgid "Languages"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:388
msgid "Locales"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:422
msgid "WELCOME"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:443
#, fuzzy
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Quit"
msgstr "В_ыход "

#: pyanaconda/ui/gui/spokes/welcome.glade:457
#, fuzzy
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Continue"
msgstr "П_родолжить"

#: pyanaconda/ui/gui/spokes/welcome.glade:488
msgid "Unsupported Hardware Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:546
msgid "Unsupported Hardware"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:10
#: pyanaconda/ui/gui/spokes/user.glade:346
msgid "CREATE USER"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:62
msgctxt "GUI|User"
msgid "_Full name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:80
#, fuzzy
msgctxt "GUI|User"
msgid "_User name"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/user.glade:100
#, fuzzy
msgid "Full Name"
msgstr "Имя группы"

#: pyanaconda/ui/gui/spokes/user.glade:131
#, fuzzy
msgctxt "GUI|User"
msgid "_Password"
msgstr "Пароль:"

#: pyanaconda/ui/gui/spokes/user.glade:149
#, fuzzy
msgctxt "GUI|User"
msgid "_Confirm password"
msgstr "Пароль VNC"

#: pyanaconda/ui/gui/spokes/user.glade:201
msgid ""
"<b>Tip:</b> Keep your user name shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:211
msgctxt "GUI|User"
msgid "_Require a password to use this account"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:270
msgctxt "GUI|User"
msgid "_Make this user administrator"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:289
msgctxt "GUI|User"
msgid "_Advanced..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:62
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:539
msgid "RAID0 <span foreground=\"grey\">(Performance)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:66
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:543
msgid "RAID1 <span foreground=\"grey\">(Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:70
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:547
msgid "RAID4 <span foreground=\"grey\">(Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:551
msgid "RAID5 <span foreground=\"grey\">(Distributed Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:78
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:555
msgid "RAID6 <span foreground=\"grey\">(Redundant Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:82
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:559
msgid "RAID10 <span foreground=\"grey\">(Performance, Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:181
#, fuzzy
msgid "Add a new mount point."
msgstr "Добавить точку монтирования"

#: pyanaconda/ui/gui/spokes/custom.glade:182
#, fuzzy
msgctxt "GUI|Custom Partitioning"
msgid "Add"
msgstr "_Добавить"

#: pyanaconda/ui/gui/spokes/custom.glade:197
#, fuzzy
msgid "Remove the selected mount point(s)."
msgstr "Удалить выбранные точки монтирования."

#: pyanaconda/ui/gui/spokes/custom.glade:198
#, fuzzy
msgctxt "GUI|Custom Partitioning"
msgid "Remove"
msgstr "Удалить раскладку"

#: pyanaconda/ui/gui/spokes/custom.glade:213
#, fuzzy
msgid "Reload storage configuration from disk."
msgstr "Ошибка настройки хранилища: %s"

#: pyanaconda/ui/gui/spokes/custom.glade:214
msgctxt "GUI|Custom Partitioning"
msgid "Refresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:279
#: pyanaconda/ui/gui/spokes/custom.glade:1013
#: pyanaconda/ui/gui/spokes/custom.glade:1157
#: pyanaconda/ui/gui/spokes/custom.glade:1244
msgid "Selected Device"
msgstr "Выбранное устройство"

#: pyanaconda/ui/gui/spokes/custom.glade:297
#: pyanaconda/ui/gui/spokes/custom.glade:1031
#: pyanaconda/ui/gui/spokes/custom.glade:1175
#: pyanaconda/ui/gui/spokes/custom.glade:1262
msgid "Device description"
msgstr "Описание устройства"

#: pyanaconda/ui/gui/spokes/custom.glade:358
#, fuzzy
msgid "Mount _Point:"
msgstr "<b>Точка монтирования:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:405
#, fuzzy
msgid "_Desired Capacity:"
msgstr "<b>Требуемый размер:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:456
#, fuzzy
msgid "_Label:"
msgstr "<b>Метка:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:503
#, fuzzy
msgid "Device(s):"
msgstr "Устройство:"

#: pyanaconda/ui/gui/spokes/custom.glade:533
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Modify..."
msgstr "Изменить..."

#: pyanaconda/ui/gui/spokes/custom.glade:565
#, fuzzy
msgid "_Name:"
msgstr "Имя:"

#: pyanaconda/ui/gui/spokes/custom.glade:620
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Device _Type:"
msgstr "<b>Тип устройства:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:664
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Encrypt"
msgstr "Зашифровать"

#: pyanaconda/ui/gui/spokes/custom.glade:705
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "File S_ystem:"
msgstr "Файловая система"

#: pyanaconda/ui/gui/spokes/custom.glade:740
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Ref_ormat"
msgstr "Форматировать заново"

#: pyanaconda/ui/gui/spokes/custom.glade:794
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Volume Group:"
msgstr "Группа томов:"

#: pyanaconda/ui/gui/spokes/custom.glade:848
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Modify..."
msgstr "Изменить..."

#: pyanaconda/ui/gui/spokes/custom.glade:880
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "RA_ID Level:"
msgstr "Устройство RAID"

#: pyanaconda/ui/gui/spokes/custom.glade:945
msgid "_Update Settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:968
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1065
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""
"Устройство зашифровано, его данные недоступны без пароля. Ниже его можно "
"разблокировать."

#: pyanaconda/ui/gui/spokes/custom.glade:1086
#, fuzzy
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Passphrase:"
msgstr "Пароль:"

#: pyanaconda/ui/gui/spokes/custom.glade:1114
#, fuzzy
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Unlock"
msgstr "_Разблокировать"

#: pyanaconda/ui/gui/spokes/custom.glade:1209
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""
"Это устройство не может быть изменено напрямую. Следует его удалить и "
"добавить другое устройство."

#: pyanaconda/ui/gui/spokes/custom.glade:1357
msgid "AVAILABLE SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1410
msgid "TOTAL SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1450
msgctxt "GUI|Custom Partitioning"
msgid "_Reset All"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:23
#, fuzzy
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:56
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:70
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_NIC:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:92
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use _DCB"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:108
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use auto _VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:144
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Add FCoE Disk(s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:187
msgid "Error adding FCoE SAN."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:215
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:40
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:54
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_OK"
msgstr "OК"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:86
msgid "ADD iSCSI STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:115
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:142
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Target IP Address:"
msgstr "IP-адрес"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:169
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "iSCSI _Initiator Name:"
msgstr "Необходимо указать имя iSCSI-инициатора."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:185
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:751
msgid "No credentials (discovery authentication disabled)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:186
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:752
msgid "CHAP pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:187
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:753
msgid "CHAP pair and a reverse pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:202
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Discovery Authentication Type:"
msgstr "_Использовать проверку подлинности"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:217
#, fuzzy
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"
msgstr "<span size=\"small\"><b>Пример:</b> squid.mysite.org:3128</span>"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:262
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _User name:"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:276
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Password:"
msgstr "Пароль VNC"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:382
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _User name:"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:396
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Пароль VNC"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:410
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP User _name:"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:424
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "Пароль VNC"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:457
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Start"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:494
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:534
msgid "Discovery login rejected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:549
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check "
"your authorization information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:571
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Retry"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:611
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Bind targets to network interfaces"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:686
#, fuzzy
msgid "Node Name"
msgstr "Сетевое имя"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:697
msgid "Interface"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:708
msgid "Portal"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:737
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login"
msgid "_Node Login Authentication Type:"
msgstr "_Использовать проверку подлинности"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:754
msgid "Use the credentials from discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:789
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _User name:"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:803
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Password:"
msgstr "Пароль VNC"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:909
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _User name:"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:923
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Пароль VNC"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:937
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "_Reverse CHAP User name:"
msgstr "Имя пользователя"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:951
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "Пароль VNC"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:999
msgctxt "GUI|Advanced Storage|iSCSI|Login|Login"
msgid "_Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1055
#, fuzzy
msgid "Node login failed."
msgstr "Ошибка подключения"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1070
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1092
msgctxt "GUI|Advanced Storage|iSCSI|Login|Retry"
msgid "Retry _Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:22
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:36
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_OK"
msgstr "OК"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:68
msgid "ADD DASD STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:97
msgid "To use DASD disks, you must provide the device number."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:125
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Device number:"
msgstr "Устройство:"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:145
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:182
msgid "Discovering DASD devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:224
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:267
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:241
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:267
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:82
msgid "ADD zFCP STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:111
msgid ""
"To use zFCP disks, you must provide the device number, WWPN, and LUN "
"configured for the device."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:139
#, fuzzy
msgctxt "GUI|Advanced Storage|zFCP|Device Number"
msgid "_Device number:"
msgstr "Устройство:"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:168
msgid "WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:188
msgctxt "GUI|Advanced Storage|zFCP|Start Discovery"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:225
msgid "Discovering zFCP devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:284
msgid ""
"The following error occurred discovering zFCP devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:310
msgctxt "GUI|Advanced Storage|zFCP|Retry Discovery"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:356
msgid "LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:26
#: pyanaconda/ui/gui/spokes/lib/cart.glade:78
msgid "SELECTED DISKS"
msgstr "ВЫБРАННЫЕ ДИСКИ"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:43
#, fuzzy
msgctxt "GUI|Selected Disks Dialog"
msgid "_Close"
msgstr "Закрыть"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:106
msgid "Boot"
msgstr "Загрузка"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:120
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:222
msgid "Description"
msgstr "Описание"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:158
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:261
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:739
msgid "Free"
msgstr "Своб."

#: pyanaconda/ui/gui/spokes/lib/cart.glade:199
#, fuzzy
msgctxt "GUI|Selected Disks Dialog"
msgid "_Remove"
msgstr "Удалить раскладку"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:226
msgid "Disk summary goes here"
msgstr "Сводная информация по диску"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:21
#, fuzzy
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:37
#, fuzzy
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Delete It"
msgstr "_Удалить"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:129
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:144
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Select"
msgstr "Ничего не выбрано"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:176
msgid "CONFIGURE MOUNT POINT"
msgstr "НАСТРОЙКА ТОЧКИ МОНТИРОВАНИЯ"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:191
msgid "Select one or more disks this device may reside on."
msgstr "Укажите диски, на которых может располагаться устройство."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:321
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:336
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Add mount point"
msgstr "_Добавить точку монтирования"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:369
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Desired Capacity:"
msgstr "<b>Требуемый размер:</b>"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:386
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Mount _Point:"
msgstr "<b>Точка монтирования:</b>"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:404
msgid "ADD A NEW MOUNT POINT"
msgstr "ДОБАВИТЬ НОВУЮ ТОЧКУ МОНТИРОВАНИЯ"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:421
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""
"Дополнительные настройки станут доступными\n"
"после создания этой точки монтирования."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:456
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""
"Эта точка монтирования уже\n"
"используется. Выбрать другую?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:480
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr "например: «20 Гб», «500 Мб» (без кавычек)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:535
msgid "Single <span foreground=\"grey\">(No Redundancy, No Striping)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:583
#, fuzzy
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:597
#, fuzzy
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Save"
msgstr "_Сохранить"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:629
#, fuzzy
msgid "CONFIGURE CONTAINER"
msgstr "НАСТРОЙКА ТОЧКИ МОНТИРОВАНИЯ"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:644
#, fuzzy
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr "Определите имя для группы томов и выберите как минимум один диск."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:662
#, fuzzy
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Name:"
msgstr "Имя:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:781
#, fuzzy
msgid "RAID Level:"
msgstr "Устройство RAID"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:809
#, fuzzy
msgid "Encrypt"
msgstr "Зашифровать"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:850
#, fuzzy
msgid "Please enter a valid name."
msgstr "Укажите правильную точку монтирования."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:880
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "Si_ze policy:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:895
msgid "Automatic"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:896
msgid "As large as possible"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:897
msgid "Fixed"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:8
msgid "UNFORMATTED DASDS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:23
#, fuzzy
msgctxt "GUI|DAD Format Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:37
#, fuzzy
msgctxt "GUI|DAD Format Dialog"
msgid "_OK"
msgstr "OК"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:70
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:86
#, fuzzy
msgid "Unformatted DASDs Detected"
msgstr "Форматирование DASD-устройства"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:136
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:160
msgctxt "GUI|DAD Format Dialog"
msgid "_Format with dasdfmt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:207
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while formatting completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:247
msgid "Disk formatting complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade:25
msgid "An unknown error occurred during installation.  Details are below."
msgstr ""
"Во время установки произошла неизвестная ошибка.  Подробности приводятся "
"ниже."

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:23
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:38
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "_Save Passphrase"
msgstr "Сохранить пароль"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:66
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr "ПАРОЛЬ ШИФРОВАНИЯ ДИСКА"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:82
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""
"Вы выбрали шифрование данных. Необходимо создать пароль доступа при запуске "
"операционной системы."

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:103
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "_Passphrase:"
msgstr "Пароль:"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:132
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "Con_firm:"
msgstr "Подтверждение:"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:270
#, fuzzy
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""
"Предупреждение: Текущая раскладка клавиатуры — <b>%s</b>. Если изменить "
"раскладку, то после уставновки это может помешать расшифровке дисков."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:8
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:71
#, fuzzy
msgid "RESCAN DISKS"
msgstr "ВЕРНУТЬ ПРОСТРАНСТВО"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:25
#, fuzzy
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:39
#, fuzzy
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_OK"
msgstr "OК"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:87
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:118
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:143
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Rescan Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:176
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:212
msgid "Scanning disks.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:223
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:254
msgid "Disk rescan complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:269
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:55
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Cancel"
msgstr "_Отмена"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:70
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Reclaim space"
msgstr "_Вернуть пространство"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:104
msgid "RECLAIM DISK SPACE"
msgstr "ВЕРНУТЬ ПРОСТРАНСТВО"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:121
msgid "Description goes here."
msgstr "Описание"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:174
#, fuzzy
msgid "File System"
msgstr "Файловая система"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:185
msgid "Reclaimable Space"
msgstr "Место для возврата"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:196
#: pyanaconda/ui/gui/spokes/lib/summary.glade:144
msgid "Action"
msgstr "Действие"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:221
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Preserve"
msgstr "Сохранить"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:237
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Delete"
msgstr "Удалить"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:253
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Shrink"
msgstr "Ужать"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:367
#, fuzzy
msgid "Reclaim"
msgstr "Перераспределить _пространство"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:25
#: pyanaconda/ui/gui/spokes/lib/summary.glade:89
msgid "SUMMARY OF CHANGES"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:42
#, fuzzy
msgctxt "GUI|Summary Dialog"
msgid "_Cancel & Return to Custom Partitioning"
msgstr "Выбрано собственное разбиение диска"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:57
#, fuzzy
msgctxt "GUI|Summary Dialog"
msgid "_Accept Changes"
msgstr "_Применить"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:105
msgid ""
"Your customizations will result in the following changes taking effect after "
"you return to the main menu and begin installation:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:133
msgid "Order"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:166
#, fuzzy
msgid "Device"
msgstr "Устройство:"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:177
#, fuzzy
msgid "Mount point"
msgstr "_Добавить точку монтирования"

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:35
msgid ""
"The system needs better quality of random data, you can improve it by typing "
"randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:53
msgid "Random data quality:"
msgstr ""

#: pyanaconda/ui/gui/main.glade:24
msgid "_Exit Installer"
msgstr "В_ыход из программы установки"

#: pyanaconda/ui/gui/main.glade:94
#, fuzzy
msgctxt "GUI|Main"
msgid "_No"
msgstr "_Нет"

#: pyanaconda/ui/gui/main.glade:108
#, fuzzy
msgctxt "GUI|Main"
msgid "_Yes"
msgstr "_Да"

#: pyanaconda/ui/gui/main.glade:151
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""
"Действительно прервать\n"
"процесс установки?"

#: pyanaconda/ui/gui/main.glade:175
#, fuzzy
msgid "Quit"
msgstr "В_ыход "

#: pyanaconda/ui/gui/hubs/summary.glade:8
#: pyanaconda/ui/gui/hubs/summary.glade:147
msgid "INSTALLATION SUMMARY"
msgstr "ОБЗОР УСТАНОВКИ"

#: pyanaconda/ui/gui/hubs/summary.glade:84
#, fuzzy
msgctxt "GUI|Summary"
msgid "_Quit"
msgstr "В_ыход "

#: pyanaconda/ui/gui/hubs/summary.glade:100
#, fuzzy
msgctxt "GUI|Summary"
msgid "_Begin Installation"
msgstr "_Начать установку"

#: pyanaconda/ui/gui/hubs/summary.glade:131
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:8
#: pyanaconda/ui/gui/hubs/progress.glade:261
msgid "CONFIGURATION"
msgstr "КОНФИГУРАЦИЯ"

#: pyanaconda/ui/gui/hubs/progress.glade:91
msgid "Preparing to install"
msgstr "Подготовка к установке"

#: pyanaconda/ui/gui/hubs/progress.glade:149
#, fuzzy
msgctxt "GUI|Progress"
msgid "_Finish configuration"
msgstr "Завершить настройку"

#: pyanaconda/ui/gui/hubs/progress.glade:202
#, fuzzy
msgctxt "GUI|Progress"
msgid "_Reboot"
msgstr "_Перезагрузить"

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install to Hard Drive"
msgstr "Установить на жёсткий диск"

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install"
msgstr "Установить"

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install the live CD to your hard disk"
msgstr "Установить Live CD на жёсткий диск."

#: /home/sbueno/scm/anaconda/data/liveinst/console.apps/liveinst:5
msgid "Starting Install to Hard Drive"
msgstr "Начинается установка на жёсткий диск"

#: data/liveinst/gnome/fedora-welcome.desktop.in.h:1
#: data/liveinst/gnome/fedora-welcome:65
msgid "Welcome to Fedora"
msgstr "Добро пожаловать в Fedora"

#: anaconda:463
#, fuzzy, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""
"%s требует %s МБ памяти для установки, но на этом компьютере доступно %s "
"МБ.\n"

#: anaconda:465
#, fuzzy, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of "
"memory, but you only have %(total_ram)s MB\n"
"."
msgstr ""
"Для графической установки %s необходимо %s МБ, но у вас есть только %s МБ."

#: anaconda:468
msgid ""
"\n"
"Press <return> to reboot your system.\n"
msgstr ""

#: anaconda:470
msgid "Not enough RAM"
msgstr "Недостаточно оперативной памяти"

#: anaconda:471
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""
" Попробуйте запустить установщик в текстовом режиме, выполнив:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" из терминала пользователя root."

#: anaconda:474
msgid " Starting text mode."
msgstr "Запуск текстового режима."

#: anaconda:511
#, fuzzy
msgid "Fatal Error"
msgstr "Ошибка"

#: anaconda:636
msgid "DISPLAY variable not set. Starting text mode."
msgstr "Переменная DISPLAY не определена. Запускается текстовый режим."

#: anaconda:644
msgid "Graphical installation is not available. Starting text mode."
msgstr "Графическая установка невозможна. Запускается текстовый режим."

#: anaconda:652
#, fuzzy
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to "
"use VNC mode instead?"
msgstr ""
"Текстовый режим предоставляет ограниченный выбор параметров установки. Он не "
"позволит вам указать ваш собственный метод разбиения на разделы или "
"выбранные пакеты. Хотите ли вы использовать режим VNC вместо этого?"

#: anaconda:742
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr "Для начала установки выполните ssh install@%s."

#: anaconda:744
msgid "Please ssh install@<host> to continue installation."
msgstr "Для продолжения установки выполните ssh install@<узел>."

#: anaconda:985
msgid "Unable to create PID file"
msgstr ""

#: anaconda:986
#, python-format
msgid ""
"Anaconda is unable to create %s because the file already exists. Anaconda is "
"already running, or a previous instance of anaconda has crashed."
msgstr ""

#: data/liveinst/gnome/fedora-welcome:94
msgid "Try Fedora"
msgstr "Попробуйте Fedora"

#: data/liveinst/gnome/fedora-welcome:110
msgid ""
"You are currently running Fedora from live media.\n"
"You can install Fedora now, or choose \"Install to Hard Drive\" in the "
"Activities Overview at any later time."
msgstr ""
"Fedora запущена с установочного диска.\n"
"Вы можете установить Fedora сейчас или в любое время выбрать «Установить на "
"жесткий диск» в меню «Обзор»."

#: data/liveinst/gnome/fedora-welcome:128
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""
"Вы можете в любое время выбрать «Установить на жесткий диск»\n"
"в пункте «Обзор»."

#: data/liveinst/gnome/fedora-welcome:131
msgid "Close"
msgstr "Закрыть"

#~ msgid ""
#~ "Your system had dirty file systems which you chose not to mount.  Press "
#~ "return to get a shell from which you can fsck and mount your partitions. "
#~ "%s"
#~ msgstr ""
#~ "В системе есть «грязные» файловые системы, которые не были выбраны для "
#~ "монтирования. Нажмите return для входа в оболочку, где можно будет "
#~ "выполнить проверку fsck и смонтировать разделы. %s"

#~ msgid "BTRFS"
#~ msgstr "BTRFS"

#~ msgid "_Fedora"
#~ msgstr "_Fedora"

#~ msgid "Red Hat Enterprise Linux"
#~ msgstr "Red Hat Enterprise Linux"

#~ msgid "c"
#~ msgstr "c"

#, fuzzy
#~ msgid "Failed to set up installation source. Check the source address."
#~ msgstr ""
#~ "Не удалось настроить источник для установки, проверьте адрес репозитория"

#, fuzzy
#~ msgid ""
#~ "You have provided a weak password: %s\n"
#~ "Would you like to use it anyway?"
#~ msgstr ""
#~ "Указанный пароль слишком прост: %s.  Нажмите «Готово» ещё раз, чтобы всё "
#~ "равно его использовать."

#, fuzzy
#~ msgid ""
#~ "You have provided a weak password.\n"
#~ "Would you like to use it anyway?"
#~ msgstr ""
#~ "Указанный пароль слишком прост: %s.  Нажмите «Готово» ещё раз, чтобы всё "
#~ "равно его использовать."

#, fuzzy
#~ msgctxt "GUI|Spoke"
#~ msgid "DATE & _TIME"
#~ msgstr "ДАТА И ВРЕМЯ"

#, fuzzy
#~ msgctxt "GUI|Installation Destination|Filter|zSeries"
#~ msgid "_Label"
#~ msgstr "<b>Метка:</b>"

#, fuzzy
#~ msgctxt "GUI|Installation Destination|Filter|zSeries"
#~ msgid "_Go"
#~ msgstr "_Нет"

#, fuzzy
#~ msgctxt "GUI|Network|Wired"
#~ msgid "C_onfigure..."
#~ msgstr "Н_астроить..."

#, fuzzy
#~ msgctxt "GUI|Software Source|Proxy Dialog"
#~ msgid "_Ok"
#~ msgstr "OК"

#, fuzzy
#~ msgctxt "GUI|Software Source|Proxy Dialog"
#~ msgid "Pass_word"
#~ msgstr "Пароль:"

#~ msgid "Filesystem"
#~ msgstr "Файловая система"

#, fuzzy
#~ msgid "Device Name"
#~ msgstr "Устройство:"

#~ msgid ""
#~ "Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or "
#~ "'0-9'"
#~ msgstr "Имя узла должно начинаться с символов a-z, A-Z, 0-9."

#~ msgid "You need more space on the following file systems:\n"
#~ msgstr "Требуется больше места на следующих файловых системах:\n"

#~ msgid "Can't have a question in command line mode!"
#~ msgstr "Нельзя задавать вопрос в режиме командной строки!"

#, fuzzy
#~ msgid "That name is invalid. Try something else?"
#~ msgstr "Это неверная точка монтирования. Попробовать ещё?"

#~ msgid "Error downloading package metadata..."
#~ msgstr "Ошибка загрузки метаданных пакета..."

#, fuzzy
#~ msgid "Nothing here yet."
#~ msgstr "Ничего не выбрано"

#, fuzzy
#~ msgid "Firmware _RAID"
#~ msgstr "Прошивка не найдена"

#, fuzzy
#~ msgid "<b>Authentication required by wireless network</b>"
#~ msgstr "Требуется проверка подлинности"

#~ msgid "column"
#~ msgstr "колонка"

#, fuzzy
#~ msgid "<b>Partition scheme:</b>"
#~ msgstr "<b>Тип раздела:</b>"

#, fuzzy
#~ msgid "<b>_Proxy URL</b>"
#~ msgstr "<b>Адрес прокси</b>"

#, fuzzy
#~ msgid "<b>User_name</b>"
#~ msgstr "<b>Имя пользователя</b>"

#, fuzzy
#~ msgid "<b>Pass_word</b>"
#~ msgstr "<b>Пароль</b>"

#, fuzzy
#~ msgid "Invalid repository name. Try something else?"
#~ msgstr "Это неверная точка монтирования. Попробовать ещё?"

#, fuzzy
#~ msgid "<b>_Label:</b>"
#~ msgstr "<b>Метка:</b>"

#, fuzzy
#~ msgid "<b>File _System:</b>"
#~ msgstr "<b>Файловая система:</b>"

#, fuzzy
#~ msgid "R_AID Level:"
#~ msgstr "Устройство RAID"

#~ msgid "Configure selected mountpoint."
#~ msgstr "Настроить точку монтирования."

#~ msgid "<span size=\"small\" foreground=\"white\">AVAILABLE SPACE</span>"
#~ msgstr ""
#~ "<span size=\"small\" foreground=\"white\">ДОСТУПНОЕ ПРОСТРАНСТВО</span>"

#~ msgid "<span size=\"small\" foreground=\"white\">TOTAL SPACE</span>"
#~ msgstr "<span size=\"small\" foreground=\"white\">ВСЕГО ПРОСТРАНСТВА</span>"

#~ msgid "Completed"
#~ msgstr "Выполнено"

#~ msgid ""
#~ "(%s)\n"
#~ "Command line mode requires all choices to be specified in a kickstart "
#~ "configuration file."
#~ msgstr ""
#~ "(%s)\n"
#~ "При установке в режиме командной строки все ответы на вопросы программы "
#~ "установки должны быть указаны в файле конфигурации кикстарта."

#~ msgid ""
#~ "The following error was found while parsing the kickstart configuration "
#~ "file:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "Ошибка при разборе конфигурации кикстарта:\n"
#~ "\n"
#~ "%s"

#~ msgid ""
#~ "(%s)\n"
#~ "%s"
#~ msgstr ""
#~ "(%s)\n"
#~ "%s"

#~ msgid "In interactive step can't continue. (%s)"
#~ msgstr "Остановка на интерактивном шаге. (%s)"

#~ msgid ""
#~ "An unhandled exception has occurred.  This is most likely a bug.  Please "
#~ "save a copy of the detailed exception and file a bug report"
#~ msgstr ""
#~ "Произошло необработанное исключение. Скорее всего, это ошибка. Сохраните "
#~ "текст исключения и создайте отчёт об ошибке"

#~ msgid " with the provider of this software."
#~ msgstr " производителю этого программного компонента."

#~ msgid " against anaconda at %s"
#~ msgstr " относительно Anaconda (%s)"

#~ msgid "Install on System"
#~ msgstr "Установка в системе"

#~ msgid "Exit installer"
#~ msgstr "Прервать установку"

#~ msgid "_Exit installer"
#~ msgstr "В_ыход из программы установки"

#~ msgid ""
#~ "After shutdown, please perform a manual IPL from %s to continue "
#~ "installation."
#~ msgstr ""
#~ "После выключения системы, пожалуйста, выполните вручную IPL-загрузку с "
#~ "%s, чтобы продолжить установку."

#~ msgid "IP address is missing."
#~ msgstr "Не указан IP-адрес."

#~ msgid ""
#~ "IPv4 addresses must contain four numbers between 0 and 255, separated by "
#~ "periods."
#~ msgstr ""
#~ "Адреса в формате IPv4 должны состоять из четырёх чисел в диапазоне от 0 "
#~ "до 255, разделённых точками."

#~ msgid "'%s' is not a valid IPv6 address."
#~ msgstr "Недопустимый адрес IPv6: %s"

#~ msgid "'%s' is an invalid IP address."
#~ msgstr "Недопустимый адрес IP: %s"

#~ msgid "bootloader device"
#~ msgstr "устройство загрузчика"

#~ msgid "First sector of boot partition"
#~ msgstr "Первый сектор загрузочного раздела"

#~ msgid "Master Boot Record"
#~ msgstr "Главная загрузочная запись"

#~ msgid "EFI System Partition"
#~ msgstr "Системный раздел EFI"

#~ msgid "Apple EFI Boot Partition"
#~ msgstr "Раздел Apple EFI Boot"

#~ msgid "PReP Boot Partition"
#~ msgstr "Раздел PReP Boot"

#~ msgid "Apple Bootstrap Partition"
#~ msgstr "Раздел Apple Bootstrap"

#~ msgid "DASD"
#~ msgstr "DASD"

#~ msgid "zFCP"
#~ msgstr "zFCP"

#~ msgid "U-Boot Partition"
#~ msgstr "Раздел U-Boot"

#~ msgid "%s MB"
#~ msgstr "%s МБ"

#~ msgid "%s KB"
#~ msgstr "%s КБ"

#~ msgid "%s Byte"
#~ msgid_plural "%s Bytes"
#~ msgstr[0] "%s байт"
#~ msgstr[1] "%s байта"
#~ msgstr[2] "%s байт"

#~ msgid "<b>Upgrading %(pkgStr)s</b> (%(size)s)\n"
#~ msgstr "<b>Обновление %(pkgStr)s</b> (%(size)s)\n"

#~ msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
#~ msgstr "<b>Установка %(pkgStr)s</b> (%(size)s)\n"

#~ msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
#~ msgid_plural "Packages completed: %(donepkgs)d of %(numpkgs)d"
#~ msgstr[0] "Установлено пакетов: %(donepkgs)d из %(numpkgs)d"
#~ msgstr[1] "Установлено пакетов: %(donepkgs)d из %(numpkgs)d"
#~ msgstr[2] "Установлено пакетов: %(donepkgs)d из %(numpkgs)d"

#~ msgid "<b>Cleaning up %s</b>"
#~ msgstr "<b>Очистка %s</b>"

#~ msgid "cpio"
#~ msgstr "cpio"

#~ msgid "unpack"
#~ msgstr "распаковать"

#~ msgid "script"
#~ msgstr "скрипт"

#~ msgid "Error Installing Package"
#~ msgstr "Ошибка установки пакета"

#~ msgid ""
#~ "A %s error occurred when installing the %s package.  This could indicate "
#~ "errors when reading the installation media.  Installation cannot continue."
#~ msgstr ""
#~ "Произошла ошибка %s при установке пакета %s. Возможная причина -- ошибки "
#~ "при чтении установочного носителя. Продолжение установки невозможно."

#~ msgid "Wrong Disc"
#~ msgstr "Неверный диск"

#~ msgid "That's not the correct %s disc."
#~ msgstr "Неверный диск %s."

#~ msgid "Repository %r is missing name in configuration, using id"
#~ msgstr ""
#~ "В конфигурации отсутствует имя репозитория %r. Вместо имени будет "
#~ "использоваться идентификатор."

#~ msgid "No Network Available"
#~ msgstr "Сеть недоступна"

#~ msgid ""
#~ "Some of your software repositories require networking, but there was an "
#~ "error enabling the network on your system."
#~ msgstr ""
#~ "Некоторые репозитории требуют наличия подключения к сети, но произошла "
#~ "ошибка при ее настройке."

#~ msgid "Re_boot"
#~ msgstr "_Перезагрузка"

#~ msgid "_Eject"
#~ msgstr "_Извлечь CD"

#~ msgid "_Retry"
#~ msgstr "_Повторить"

#~ msgid ""
#~ "The file %s cannot be opened.  This is due to a missing file, a corrupt "
#~ "package or corrupt media.  Please verify your installation source.\n"
#~ "\n"
#~ "If you exit, your system will be left in an inconsistent state that will "
#~ "likely require reinstallation.\n"
#~ "\n"
#~ msgstr ""
#~ "Не удается открыть файл %s. Возможная причина -- отсутствие файла или "
#~ "сбойный пакет/ носитель. Убедитесь, что дерево установки содержит все "
#~ "необходимые пакеты.\n"
#~ "\n"
#~ "При отмене установки система останется в нестабильном состоянии, "
#~ "вероятно, требующем переустановки.\n"
#~ "\n"

#~ msgid "Retrying"
#~ msgstr "Повторная попытка"

#~ msgid "Retrying download."
#~ msgstr "Повтор попытки загрузки."

#~ msgid ""
#~ "There was an error running your transaction for the following reason: %s\n"
#~ msgstr ""
#~ "При выполнении транзакции произошла ошибка по следующей причине: %s\n"

#~ msgid "_Back"
#~ msgstr "_Назад"

#~ msgid ""
#~ "There was an error running your transaction for the following reason: "
#~ "%s.\n"
#~ msgstr ""
#~ "В ходе вашей транзакции произошла ошибка по следующей причине: %s.\n"

#~ msgid "Error Running Transaction"
#~ msgstr "Ошибка при выполнении транзакции"

#~ msgid "file conflicts"
#~ msgstr "конфликты файлов"

#~ msgid "older package(s)"
#~ msgstr "старые пакеты"

#~ msgid "insufficient disk space"
#~ msgstr "недостаточно места на диске"

#~ msgid "insufficient disk inodes"
#~ msgstr "недостаточное кол-во индексных дескрипторов на диске"

#~ msgid "package conflicts"
#~ msgstr "конфликты пакетов"

#~ msgid "package already installed"
#~ msgstr "пакет уже установлен"

#~ msgid "required package"
#~ msgstr "требуется пакет"

#~ msgid "package for incorrect arch"
#~ msgstr "пакет для не той архитектуры"

#~ msgid "package for incorrect os"
#~ msgstr "пакет для не той операционной системы"

#~ msgid ""
#~ "There were file conflicts when checking the packages to be installed:\n"
#~ "%s\n"
#~ msgstr ""
#~ "При проверке пакетов для установки обнаружены конфликты файлов:\n"
#~ "%s\n"

#~ msgid ""
#~ "There was an error running your transaction for the following reason(s): "
#~ "%s.\n"
#~ msgstr ""
#~ "При выполнении транзакции произошла ошибка по следующим причинам: %s\n"

#~ msgid ""
#~ "Unable to read group information from repositories.  This is a problem "
#~ "with the generation of your install tree."
#~ msgstr ""
#~ "Невозможно прочесть информацию о группах из репозиториев. Это проблема "
#~ "относится к генерированию дерева установки."

#~ msgid "Edit"
#~ msgstr "Изменить"

#~ msgid ""
#~ "Unable to read package metadata. This may be due to a missing repodata "
#~ "directory.  Please ensure that your install tree has been correctly "
#~ "generated.\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "Не удаётся прочитать метаданные пакета. Возможная причина — отсутствие "
#~ "каталога repodata. Убедитесь, что дерево установки сформировано "
#~ "правильно.\n"
#~ "\n"
#~ "%s"

#~ msgid ""
#~ "Some of the packages you have selected for install are missing "
#~ "dependencies.  You can exit the installation, go back and change your "
#~ "package selections, or continue installing these packages without their "
#~ "dependencies.  If you continue, these packages may not work correctly due "
#~ "to missing components."
#~ msgstr ""
#~ "Отсутствуют зависимости для некоторых выбранных для установки пакетов.  "
#~ "Можно выйти из процесса установки, вернуться назад и изменить набор "
#~ "пакетов или продолжить установку этих пакетов без их зависимостей. При "
#~ "продолжении установки выбранные пакеты могут работать неверно из-за "
#~ "отсутствия некоторых компонентов."

#~ msgid ""
#~ "Your selected packages require %d MB of free space for installation, but "
#~ "you do not have enough available.  You can change your selections or exit "
#~ "the installer."
#~ msgstr ""
#~ "Выбранные пакеты требуют для установки %d МБ свободного места, а у вас "
#~ "его недостаточно. Вы можете изменить выбор или выйти из программы "
#~ "установки."

#~ msgid "Post Upgrade"
#~ msgstr "После обновления"

#~ msgid "Performing post-upgrade configuration"
#~ msgstr "Выполняется настройка после обновления"

#~ msgid "Post Installation"
#~ msgstr "После установки"

#~ msgid "Performing post-installation configuration"
#~ msgstr "Выполняется настройка после установки"

#~ msgid "Installation Starting"
#~ msgstr "Запуск установки"

#~ msgid "Starting installation process"
#~ msgstr "Запуск процесса установки"

#~ msgid "Dependency Check"
#~ msgstr "Проверка зависимостей"

#~ msgid "Checking dependencies in packages selected for installation"
#~ msgstr "Проверяются зависимости выбранных для установки пакетов"

#~ msgid "Retrieving installation information."
#~ msgstr "Получение установочной информации."

#~ msgid "Retrieving installation information for %s."
#~ msgstr "Получение установочной информации для %s."

#~ msgid ""
#~ "The default installation of %s includes a set of software applicable for "
#~ "general internet usage. You can optionally select a different set of "
#~ "software now."
#~ msgstr ""
#~ "Стандартная установка %s включает набор программ, подходящих для решения "
#~ "обычных задач работы с Интернетом. Поддержку каких дополнительных задач "
#~ "вы хотели бы включить?"

#~ msgid "Graphical Desktop"
#~ msgstr "Графический рабочий стол"

#~ msgid "Software Development"
#~ msgstr "Разработка программного обеспечения"

#~ msgid "Minimal"
#~ msgstr "Минимальный набор программ"

#~ msgid ""
#~ "The default installation of %s is a minimum install. You can optionally "
#~ "select a different set of software now."
#~ msgstr ""
#~ "Стандартная установка %s включает минимальный набор программ. По желанию "
#~ "можно выбрать другой набор программ."

#~ msgid "Upgrading"
#~ msgstr "Обновление"

#~ msgid "This partition is holding the data for the hard drive install."
#~ msgstr "Этот раздел содержит данные для установки с жесткого диска."

#~ msgid "You cannot delete a partition of a LDL formatted DASD."
#~ msgstr "Невозможно удалить DASD, отформатированный как LDL"

#~ msgid "This device is part of the RAID device %s."
#~ msgstr "Это устройство является частью устройства RAID %s."

#~ msgid "This device is part of a RAID device."
#~ msgstr "Это устройство является частью устройства RAID."

#~ msgid "This device is part of an inconsistent LVM Volume Group."
#~ msgstr "Это устройство является частью рассогласованной группы томов LVM."

#~ msgid "This device is part of the LVM volume group '%s'."
#~ msgstr "Это устройство является частью группы томов LVM «%s»."

#~ msgid "This device is part of a LVM volume group."
#~ msgstr "Это устройство является частью группы томов LVM."

#~ msgid ""
#~ "This device is an extended partition which contains logical partitions "
#~ "that cannot be deleted:\n"
#~ "\n"
#~ msgstr ""
#~ "Этот раздел является расширенным и содержит логические разделы, которые "
#~ "невозможно удалить:\n"
#~ "\n"

#~ msgid ""
#~ "Installing on a USB device.  This may or may not produce a working system."
#~ msgstr ""
#~ "Установка на USB-устройство. Существует вероятность получения нерабочей "
#~ "системы."

#~ msgid ""
#~ "Installing on a FireWire device.  This may or may not produce a working "
#~ "system."
#~ msgstr ""
#~ "Установка на FireWire-устройство. Существует вероятность получения "
#~ "нерабочей системы."

#~ msgid "you have not created a bootloader stage1 target device"
#~ msgstr "вы не установили устройство 1го этапа загрузчика"

#~ msgid "Linux on %s"
#~ msgstr "Linux на %s"

#~ msgid "Unknown Linux"
#~ msgstr "Неизвестный Linux"

#~ msgid "%(product)s Linux %(version)s for %(arch)s"
#~ msgstr "%(product)s Linux %(version)s для %(arch)s"

#~ msgid "Preparing %d DASD device for use with Linux..."
#~ msgid_plural "Preparing %d DASD devices for use with Linux..."
#~ msgstr[0] "Подготовка %d DASD-устройства для использования с Linux..."
#~ msgstr[1] "Подготовка %d DASD-устройств для использования с Linux..."
#~ msgstr[2] "Подготовка %d DASD-устройств для использования с Linux..."

#~ msgid "Resizing filesystem on %(device)s"
#~ msgstr "Изменение размера файловой системы на %(device)s"

#~ msgid "Migrating filesystem on %(device)s"
#~ msgstr "Перенос файловой системы на %(device)s"

#~ msgid "vginfo failed for %s"
#~ msgstr "сбой вызова vginfo для %s"

#~ msgid "lvs failed for %s"
#~ msgstr "сбой вызова lvs для %s"

#~ msgid "A RAID%(raidLevel)d set requires at least %(minMembers)d member"
#~ msgid_plural ""
#~ "A RAID%(raidLevel)d set requires at least %(minMembers)d members"
#~ msgstr[0] "RAID%(raidLevel)d требуется как минимум %(minMembers)d член"
#~ msgstr[1] "RAID%(raidLevel)d требуется как минимум %(minMembers)d членов"
#~ msgstr[2] "RAID%(raidLevel)d требуется как минимум %(minMembers)d членов"

#~ msgid "Back"
#~ msgstr "Назад"

#~ msgid "Confirm"
#~ msgstr "Подтвердить"

#~ msgid ""
#~ "Are you sure you want to skip entering a passphrase for device %s?\n"
#~ "\n"
#~ "If you skip this step the device's contents will not be available during "
#~ "installation."
#~ msgstr ""
#~ "Вы действительно хотите пропустить ввод пароля для устройства %s?\n"
#~ "\n"
#~ "Если вы пропустите этот шаг, содержимое устройства будет недоступно в "
#~ "процессе установки."

#~ msgid "FCoE not available"
#~ msgstr "FCoE недоступен"

#~ msgid "Unknown return code: %d."
#~ msgstr "Неизвестный код возврата: %d."

#~ msgid "%(type)s filesystem check failure on %(device)s: "
#~ msgstr "Ошибка проверки файловой системы %(type)s на %(device)s: "

#~ msgid "File system errors left uncorrected."
#~ msgstr "Ошибки файловой системы остались неисправленными."

#~ msgid "Usage or syntax error."
#~ msgstr "Ошибка использования или синтаксиса."

#~ msgid "e2fsck cancelled by user request."
#~ msgstr "Отмена e2fsck по требованию пользователя."

#~ msgid "Shared library error."
#~ msgstr "Ошибка разделяемой библиотеки."

#~ msgid ""
#~ "Recoverable errors have been detected or dosfsck has discovered an "
#~ "internal inconsistency."
#~ msgstr ""
#~ "Обнаружены некритические ошибки или dosfsck обнаружила внутреннюю "
#~ "противоречивость."

#~ msgid "Usage error."
#~ msgstr "Ошибка использования."

#~ msgid "Encrypted"
#~ msgstr "Зашифровано"

#~ msgid "Unable to change iSCSI initiator name once set"
#~ msgstr "Невозможно изменить установленное имя iSCSI-инициатора."

#~ msgid "iSCSI not available"
#~ msgstr "iSCSI недоступен"

#~ msgid "No initiator name set"
#~ msgstr "Не задано имя iSCSI-инициатора"

#~ msgid "Logging in to iSCSI node"
#~ msgstr "Вход на iSCSI-узел"

#~ msgid "Logging in to iSCSI node %s"
#~ msgstr "Вход на iSCSI-узел %s"

#~ msgid "No iSCSI nodes discovered"
#~ msgstr "iSCSI-узлы не обнаружены"

#~ msgid "No new iSCSI nodes discovered"
#~ msgstr "Не обнаружено новых iSCSI-узлов"

#~ msgid "Could not log in to any of the discovered nodes"
#~ msgstr "Не удалось войти ни в один из обнаруженных узлов"

#~ msgid "You have not specified a device number or the number is invalid"
#~ msgstr "Номер устройства не задан или задан неправильно"

#~ msgid "You have not specified a worldwide port name or the name is invalid."
#~ msgstr "Общее имя порта устройства не задано или задано неправильно."

#~ msgid "You have not specified a FCP LUN or the number is invalid."
#~ msgstr "FCP LUN не задан, или номер указан неправильно."

#~ msgid "zFCP device %s not found, not even in device ignore list."
#~ msgstr ""
#~ "zFCP-устройство %s не найдено. Его нет даже в списке игнорируемых "
#~ "устройств."

#~ msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
#~ msgstr ""
#~ "Не удалось перевести zFCP-устройство %(devnum)s в рабочий режим (%(e)s)."

#~ msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "Не удалось добавить WWPN %(wwpn)s к zFCP-устройству %(devnum)s (%(e)s)."

#~ msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
#~ msgstr "WWPN %(wwpn)s не найден на zFCP-устройстве %(devnum)s."

#~ msgid ""
#~ "Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s "
#~ "(%(e)s)."
#~ msgstr ""
#~ "Не удалось добавить LUN %(fcplun)s к WWPN %(wwpn)s на zFCP-устройстве "
#~ "%(devnum)s (%(e)s)."

#~ msgid ""
#~ "LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already "
#~ "configured."
#~ msgstr ""
#~ "LUN %(fcplun)s у WWPN %(wwpn)s на zFCP-устройстве %(devnum)s уже настроен."

#~ msgid ""
#~ "Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on "
#~ "zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "Не удалось прочитать атрибут «failed» c LUN %(fcplun)s у WWPN %(wwpn)s на "
#~ "zFCP-устройстве %(devnum)s (%(e)s)."

#~ msgid ""
#~ "Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
#~ "again."
#~ msgstr ""
#~ "Находящийся в состоянии сбоя LUN %(fcplun)s у WWPN %(wwpn)s на zFCP-"
#~ "устройстве %(devnum)s снова отключён."

#~ msgid ""
#~ "Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s "
#~ "%(fcplun)s (%(e)s)."
#~ msgstr ""
#~ "Не удалось корректно удалить SCSI-устройство, относящееся к zFCP "
#~ "%(devnum)s %(wwpn)s %(fcplun)s (%(e)s)."

#~ msgid ""
#~ "Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device "
#~ "%(devnum)s (%(e)s)."
#~ msgstr ""
#~ "Не удалось удалить LUN %(fcplun)s с WWPN %(wwpn)s на zFCP-устройстве "
#~ "%(devnum)s (%(e)s)."

#~ msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "Не удалось удалить WWPN %(wwpn)s на zFCP-устройстве %(devnum)s (%(e)s)."

#~ msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
#~ msgstr ""
#~ "Не удалось перевести zFCP-устройство %(devnum)s в автономный режим "
#~ "(%(e)s)."

#~ msgid "Install hub"
#~ msgstr "Узел установки"

#~ msgid "STORAGE"
#~ msgstr "НОСИТЕЛИ"

#~ msgid ""
#~ "When all items marked with this icon are complete, installation will "
#~ "automatically continue."
#~ msgstr ""
#~ "Когда все элементы, отмеченные этим значком будут заполнены, установка "
#~ "автоматически продолжится."

#~ msgid "Note: pool servers may not be available all the time"
#~ msgstr "Замечание: буферные серверы могут быть не всегда доступны"

#~ msgid "Connected: %s"
#~ msgstr "Подключено: %s"

#~ msgid "Error setting root password"
#~ msgstr "Ошибка при задании пароля root"

#~ msgid "_Remove Packages"
#~ msgstr "_Удалить пакеты"

#~ msgid "Getting info about requested repository"
#~ msgstr "Получение информации о запрашиваемом репозитории"

#~ msgid "enter URL of your desired repository"
#~ msgstr "введите адрес нужного репозитория"

#~ msgid "%s of %s"
#~ msgstr "%s из %s"

#~ msgid "00"
#~ msgstr "00"

#~ msgid " "
#~ msgstr " "

#~ msgid "Choose your add-ons"
#~ msgstr "Выберите дополнения"

#~ msgid "_Add custom add-on..."
#~ msgstr "_Добавить пользовательское дополнение..."

#~ msgid "Let me customize the _partitioning of the disks instead."
#~ msgstr "Я хочу сам настроить разбиение разделов на дисках."

#~ msgid "disks size"
#~ msgstr "размер дисков"

#~ msgid "Space in selected disks reclaimable by deleting existing partitions."
#~ msgstr ""
#~ "Место на выбранных дисках можно высвободить путём удаления существующих "
#~ "разделов."

#~ msgid "I don't need help; let me customize disk _partitioning."
#~ msgstr "Помощь не нужна. Настройка _разделов будет выполнена вручную"

#~ msgid "ADD A CUSTOM ADD-ON"
#~ msgstr "ДОБАВИТЬ ВЫБРАННЫЕ ДОПОЛНЕНИЯ"

#~ msgid "Where is the yum repository for your add-on?"
#~ msgstr "Где расположен репозиторий yum для этого дополнения?"

#~ msgid "Repository"
#~ msgstr "Репозиторий"

#~ msgid "Standard YUM repository:"
#~ msgstr "Стандартный репозиторий YUM:"

#~ msgid "eg: \"/\", \"/home\", \"swap\" (minus the quotation marks)"
#~ msgstr "напр.: «/», «/home», «swap» (без кавычек)"

#~ msgid "_Back to destination selection"
#~ msgstr "_Назад к выбору получателя"

#~ msgid "<b>Desired Capacity (MB):</b>"
#~ msgstr "<b>Требуемый размер (Мб):</b>"

#~ msgid "Redundancy (mirror)"
#~ msgstr "Избыточность (зеркало)"

#~ msgid "(-0Gb)"
#~ msgstr "(-0Гб)"

#~ msgid "Optimized performance (stripe)"
#~ msgstr "Оптимизированная производительность (чередование)"

#~ msgid "Error detection (parity)"
#~ msgstr "Обнаружение ошибок (четности)"

#~ msgid "Distributed"
#~ msgstr "Распределенный"

#~ msgid "Redundant"
#~ msgstr "Избыточный"

#~ msgid "_Customize..."
#~ msgstr "Настроить"

#~ msgid "addButton"
#~ msgstr "Кнопка добавления"

#~ msgid "removeButton"
#~ msgstr "Кнопка удаления"

#~ msgid "configureButton"
#~ msgstr "Кнопка настройки"

#~ msgid "CONFIGURE VOLUME GROUP"
#~ msgstr "НАСТРОЙКА ГРУППЫ ТОМОВ"

#~ msgid "Id"
#~ msgstr "Идентификатор"

#~ msgid "keymap warning label"
#~ msgstr "метка предупреждения раскладки"

#~ msgid "<b>%s disks; %s reclaimable space</b> (in filesystems)"
#~ msgstr "<b>%s дисков; место для возврата: %s</ b> (в файловых системах)"

#~ msgid "Total selected space to reclaim:  <b>%s</b>"
#~ msgstr "Всего выбрано места для возрата: <b>%s</b>"

#~ msgid "Installation requires a total of <b>%s</b> for system data."
#~ msgstr "Установка требует <b>%s</b> для системных данных."

#~ msgid "Root"
#~ msgstr "Корень"
